-- MySQL dump 10.13  Distrib 5.1.73, for redhat-linux-gnu (x86_64)
--
-- Host: localhost    Database: ped_bioinf_portal
-- ------------------------------------------------------
-- Server version	5.1.73

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `Articles`
--

DROP TABLE IF EXISTS `Articles`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Articles` (
  `ID` int(11) NOT NULL AUTO_INCREMENT,
  `PMID` varchar(20) COLLATE utf8_bin NOT NULL,
  `Title` text COLLATE utf8_bin NOT NULL,
  `Authors` text COLLATE utf8_bin NOT NULL,
  `Journal` text COLLATE utf8_bin NOT NULL,
  `Abstract` text COLLATE utf8_bin NOT NULL,
  `PubDate` date NOT NULL,
  `Ranking` int(11) NOT NULL,
  `Analysis` varchar(1000) COLLATE utf8_bin DEFAULT NULL,
  PRIMARY KEY (`ID`),
  UNIQUE KEY `PMID` (`PMID`)
) ENGINE=InnoDB AUTO_INCREMENT=337195 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Articles`
--

LOCK TABLES `Articles` WRITE;
/*!40000 ALTER TABLE `Articles` DISABLE KEYS */;
INSERT INTO `Articles` VALUES (337150,'26525104','RNA-Seq of Tumor-Educated Platelets Enables Blood-Based Pan-Cancer, Multiclass, and Molecular Pathway Cancer Diagnostics.','Best MG,Sol N,Kooi I,Tannous J,Westerman BA,Rustenburg F,Schellen P,Verschueren H,Post E,Koster J,Ylstra B,Ameziane N,Dorsman J,Smit EF,Verheul HM,Noske DP,Reijneveld JC,Nilsson RJ,Tannous BA,Wesseling P,Wurdinger T','Cancer cell','Tumor-educated blood platelets (TEPs) are implicated as central players in the systemic and local responses to tumor growth, thereby altering their RNA profile. We determined the diagnostic potential of TEPs by mRNA sequencing of 283 platelet samples. We distinguished 228 patients with localized and metastasized tumors from 55 healthy individuals with 96% accuracy. Across six different tumor types, the location of the primary tumor was correctly identified with 71% accuracy. Also, MET or HER2-positive, and mutant KRAS, EGFR, or PIK3CA tumors were accurately distinguished using surrogate TEP mRNA profiles. Our results indicate that blood platelets provide a valuable platform for pan-cancer, multiclass cancer, and companion diagnostics, possibly enabling clinical advances in blood-based &apo;liquid biopsies&apo;.\nCopyright © 2015 The Authors. Published by Elsevier Inc. All rights reserved.\n','2015-11-11',0,''),(337151,'16053509','Experimental trial for diagnosis of pancreatic ductal carcinoma based on gene expression profiles of pancreatic ductal cells.','Ishikawa M,Yoshida K,Yamashita Y,Ota J,Takada S,Kisanuki H,Koinuma K,Choi YL,Kaneda R,Iwao T,Tamada K,Sugano K,Mano H','Cancer science','Pancreatic ductal carcinoma (PDC) remains one of the most intractable human malignancies, mainly because of the lack of sensitive detection methods. Although gene expression profiling by DNA microarray analysis is a promising tool for the development of such detection systems, a simple comparison of pancreatic tissues may yield misleading data that reflect only differences in cellular composition. To directly compare PDC cells with normal pancreatic ductal cells, we purified MUC1-positive epithelial cells from the pancreatic juices of 25 individuals with a normal pancreas and 24 patients with PDC. The gene expression profiles of these 49 specimens were determined with DNA microarrays containing >44 000 probe sets. Application of both Welch&apo;s analysis of variance and effect size-based selection to the expression data resulted in the identification of 21 probe sets corresponding to 20 genes whose expression was highly associated with clinical diagnosis. Furthermore, correspondence analysis and 3-D projection with these probe sets resulted in separation of the transcriptomes of pancreatic ductal cells into distinct but overlapping spaces corresponding to the two clinical classes. To establish an accurate transcriptome-based diagnosis system for PDC, we applied supervised class prediction algorithms to our large data set. With the expression profiles of only five predictor genes, the weighted vote method diagnosed the class of samples with an accuracy of 81.6%. Microarray analysis with purified pancreatic ductal cells has thus provided a basis for the development of a sensitive method for the detection of PDC.\n','2005-08-01',0,'PCA,Tumour purity,Gene expression,Correlations'),(337152,'17699763','Zonal heterogeneity for gene expression in human pancreatic carcinoma.','Nakamura T,Kuwai T,Kitadai Y,Sasaki T,Fan D,Coombes KR,Kim SJ,Fidler IJ','Cancer research','Using Affymetrix HG-U133 Plus 2.0 array and laser capture microdissection techniques, we determined whether different zones of the same pancreatic tumor exhibited differential expression of genes. Human L3.6pl pancreatic cancer cells were implanted into the pancreas of nude mice. Three weeks later when tumors were 7 to 9 mm in diameter, gene expression patterns in tumor cells within the central and peripheral zones were compared, and 1,222 genes showed statistically significant differences. Bioinformatic functional analysis revealed that 346 up-regulated genes in the peripheral zone were related to cytoskeleton organization and biogenesis, cell cycle, cell adhesion, cell motility, DNA replication, localization, integrin-mediated signaling pathway, development, morphogenesis, and IkappaB kinase/nuclear factor-kappaB cascade; 876 up-regulated genes in the central zone were related to regulation of cell proliferation, regulation of transcription, transmembrane receptor protein tyrosine kinase signaling pathways, response to stress, small GTPase-mediated signal transduction, hexose metabolism, cell death, response to external stimulus, carbohydrate metabolism, and response to wounding. The reliability of the microarray results were confirmed by in situ hybridization analysis of the expression of two genes. Collectively, the data showed zonal heterogeneity for gene expression profiles in tumors and suggest that characterization of zonal gene expression profiles is essential if microarray analyses of genetic profiles are to produce reproducible data, predict disease prognosis, and allow design of specific therapeutics.\n','2007-08-16',0,'PCA,Tumour purity,Gene expression,Correlations'),(337153,'18413752','Coordinated epidermal growth factor receptor pathway gene overexpression predicts epidermal growth factor receptor inhibitor sensitivity in pancreatic cancer.','Jimeno A,Tan AC,Coffa J,Rajeshkumar NV,Kulesza P,Rubio-Viqueira B,Wheelhouse J,Diosdado B,Messersmith WA,Iacobuzio-Donahue C,Maitra A,Varella-Garcia M,Hirsch FR,Meijer GA,Hidalgo M','Cancer research','The epidermal growth factor receptor (EGFR) inhibitor erlotinib is approved for treatment of pancreatic cancer but the overall activity is minimal, and known predictive factors for EGFR inhibitor efficacy are infrequent in this disease. We tested the hypothesis that global activation of the EGFR pathway is predictive of EGFR inhibitor efficacy. Pancreatic cancer tumors directly xenografted at surgery were treated with the EGFR inhibitors erlotinib and cetuximab and analyzed for biological features. Two of 10 tumors were sensitive, and by global gene expression profiling with gene set enrichment analysis, the EGFR pathway was highly expressed in sensitive compared with resistant tumors. The core gene components driving EGFR pathway overexpression were pathway ligands and positive effectors. In a prospective validation, the EGFR pathway-based signature correctly predicted anti-EGFR treatment response in eight additional tumors and was not predictive of response to gemcitabine and CI1040 (a MEK inhibitor). Analysis of EGFR, KRAS, and PIK3CA mutations and gene amplification by fluorescence in situ hybridization and multiplex ligation-dependent probe amplification showed that none of these genetic abnormalities were neither predictive nor responsible for the EGFR pathway activation. Coordinated overexpression of the EGFR pathway predicts susceptibility to EGFR inhibitors in pancreatic cancer. These results suggest a phenomenon of pathway addiction and support the value of unbiased system biology approaches in drug development.\n','2008-04-16',0,'PCA,Tumour purity,Gene expression,Correlations'),(337154,'18754008','A paracrine requirement for hedgehog signalling in cancer.','Yauch RL,Gould SE,Scales SJ,Tang T,Tian H,Ahn CP,Marshall D,Fu L,Januario T,Kallop D,Nannini-Pepe M,Kotkow K,Marsters JC,Rubin LL,de Sauvage FJ','Nature','Ligand-dependent activation of the hedgehog (Hh) signalling pathway has been associated with tumorigenesis in a number of human tissues. Here we show that, although previous reports have described a cell-autonomous role for Hh signalling in these tumours, Hh ligands fail to activate signalling in tumour epithelial cells. In contrast, our data support ligand-dependent activation of the Hh pathway in the stromal microenvironment. Specific inhibition of Hh signalling using small molecule inhibitors, a neutralizing anti-Hh antibody or genetic deletion of smoothened (Smo) in the mouse stroma results in growth inhibition in xenograft tumour models. Taken together, these studies demonstrate a paracrine requirement for Hh ligand signalling in the tumorigenesis of Hh-expressing cancers and have important implications for the development of Hh pathway antagonists in cancer.\n','2008-09-18',0,'PCA,Gene expression,Correlations'),(337155,'18765825','Gene expression profiling-based identification of cell-surface targets for developing multimeric ligands in pancreatic cancer.','Balagurunathan Y,Morse DL,Hostetter G,Shanmugam V,Stafford P,Shack S,Pearson J,Trissal M,Demeure MJ,Von Hoff DD,Hruby VJ,Gillies RJ,Han H','Molecular cancer therapeutics','Multimeric ligands are ligands that contain multiple binding domains that simultaneously target multiple cell-surface proteins. Due to cooperative binding, multimeric ligands can have high avidity for cells (tumor) expressing all targeting proteins and only show minimal binding to cells (normal tissues) expressing none or only some of the targets. Identifying combinations of targets that concurrently express in tumor cells but not in normal cells is a challenging task. Here, we describe a novel approach for identifying such combinations using genome-wide gene expression profiling followed by immunohistochemistry. We first generated a database of mRNA gene expression profiles for 28 pancreatic cancer specimens and 103 normal tissue samples representing 28 unique tissue/cell types using DNA microarrays. The expression data for genes that encode proteins with cell-surface epitopes were then extracted from the database and analyzed using a novel multivariate rule-based computational approach to identify gene combinations that are expressed at an efficient binding level in tumors but not in normal tissues. These combinations were further ranked according to the proportion of tumor samples that expressed the sets at efficient levels. Protein expression of the genes contained in the top ranked combinations was confirmed using immunohistochemistry on a pancreatic tumor tissue and normal tissue microarrays. Coexpression of targets was further validated by their combined expression in pancreatic cancer cell lines using immunocytochemistry. These validated gene combinations thus encompass a list of cell-surface targets that can be used to develop multimeric ligands for the imaging and treatment of pancreatic cancer.\n','2008-09-15',0,''),(337156,'18852131','Antitumor activity and molecular effects of the novel heat shock protein 90 inhibitor, IPI-504, in pancreatic cancer.','Song D,Chaerkady R,Tan AC,García-García E,Nalli A,Suárez-Gauthier A,López-Ríos F,Zhang XF,Solomon A,Tong J,Read M,Fritz C,Jimeno A,Pandey A,Hidalgo M','Molecular cancer therapeutics','Targeting Hsp90 is an attractive strategy for anticancer therapy because the diversity and relevance of biological processes are regulated by these proteins in most cancers. However, the role and mode of action of Hsp90 inhibitors in pancreatic cancer has not been studied. This study aimed to assess the antitumor activity of the Hsp90 inhibitor, IPI-504, in pancreatic cancer and to determine the biological effects of the agent. In vitro, we show that pharmacologic inhibition of Hsp90 by IPI-504 exerts antiproliferative effects in a panel of pancreatic cancer cells in a dose- and time-dependent manner. In pancreatic cancer xenografts obtained directly from patients with pancreas cancer, the agent resulted in a marked suppression of tumor growth. Although known Hsp90 client proteins were significantly modulated in IPI-504-treated cell line, no consistent alteration of these proteins was observed in vivo other than induction of Hsp70 expression in the treated xenografted tumors. Using a proteomic profiling analysis with isotope tags for relative and absolute quantitation labeling technique, we have identified 20 down-regulated proteins and 42 up-regulated proteins on IPI-504 treatment.tumor growth Identical changes were observed in the expression of the genes coding for these proteins in a subset of proteins including HSPA1B, LGALS3, CALM1, FAM84B, FDPS, GOLPH2, HBA1, HIST1H1C, HLA-B, and MARCKS. The majority of these proteins belong to the functional class of intracellular signal transduction, immune response, cell growth and maintenance, transport, and metabolism. In summary, we show that IPI-504 has potent antitumor activity in pancreatic cancer and identify potential pharmacologic targets using a proteomics and gene expression profiling.\n','2008-10-14',0,'PCA,Tumour purity,Gene expression,Correlations'),(337157,'19260470','Combined gene expression analysis of whole-tissue and microdissected pancreatic ductal adenocarcinoma identifies genes specifically overexpressed in tumor epithelia.','Badea L,Herlea V,Dima SO,Dumitrascu T,Popescu I','Hepato-gastroenterology','The precise details of pancreatic ductal adenocarcinoma (PDAC) pathogenesis are still insufficiently known, requiring the use of high-throughput methods. However, PDAC is especially difficult to study using microarrays due to its strong desmoplastic reaction, which involves a hyperproliferating stroma that effectively &apo;masks&apo; the contribution of the minoritary neoplastic epithelial cells. Thus it is not clear which of the genes that have been found differentially expressed between normal and whole tumor tissues are due to the tumor epithelia and which simply reflect the differences in cellular composition. To address this problem, laser microdissection studies have been performed, but these have to deal with much smaller tissue sample quantities and therefore have significantly higher experimental noise.\nIn this paper we combine our own large sample whole-tissue study with a previously published smaller sample microdissection study by Grützmann et al. to identify the genes that are specifically overexpressed in PDAC tumor epithelia.\nThe overlap of this list of genes with other microarray studies of pancreatic cancer as well as with the published literature is impressive. Moreover, we find a number of genes whose over-expression appears to be inversely correlated with patient survival: keratin 7, laminin gamma 2, stratifin, platelet phosphofructokinase, annexin A2, MAP4K4 and OACT2 (MBOAT2), which are all specifically upregulated in the neoplastic epithelia, rather than the tumor stroma.\nWe improve on other microarray studies of PDAC by putting together the higher statistical power due to a larger number of samples with information about cell-type specific expression and patient survival.\n','2009-03-05',0,'PCA,Tumour purity,Gene expression,Correlations'),(337158,'19732725','FKBP51 affects cancer cell response to chemotherapy by negatively regulating Akt.','Pei H,Li L,Fridley BL,Jenkins GD,Kalari KR,Lingle W,Petersen G,Lou Z,Wang L','Cancer cell','Akt is a central regulator of cell growth. Its activity can be negatively regulated by the phosphatase PHLPP that specifically dephosphorylates the hydrophobic motif of Akt (Ser473 in Akt1). However, how PHLPP is targeted to Akt is not clear. Here we show that FKBP51 (FK506-binding protein 51) acts as a scaffolding protein for Akt and PHLPP and promotes dephosphorylation of Akt. Furthermore, FKBP51 is downregulated in pancreatic cancer tissue samples and several cancer cell lines. Decreased FKBP51 expression in cancer cells results in hyperphosphorylation of Akt and decreased cell death following genotoxic stress. Overall, our findings identify FKBP51 as a negative regulator of the Akt pathway, with potentially important implications for cancer etiology and response to chemotherapy.\n','2009-09-07',0,'PCA,Tumour purity,Gene expression,Correlations'),(337159,'19931263','Salivary transcriptomic biomarkers for detection of resectable pancreatic cancer.','Zhang L,Farrell JJ,Zhou H,Elashoff D,Akin D,Park NH,Chia D,Wong DT','Gastroenterology','Lack of detection technology for early pancreatic cancer invariably leads to a typical clinical presentation of incurable disease at initial diagnosis. New strategies and biomarkers for early detection are sorely needed. In this study, we have conducted a prospective sample collection and retrospective blinded validation to evaluate the performance and translational utilities of salivary transcriptomic biomarkers for the noninvasive detection of resectable pancreatic cancer.\nThe Affymetrix HG U133 Plus 2.0 Array (Affymetrix, Santa Clara, CA) was used to profile transcriptomes and discover altered gene expression in saliva supernatant. Biomarkers discovered from the microarray study were subjected to clinical validation using an independent sample set of 30 pancreatic cancer patients, 30 chronic pancreatitis patients, and 30 healthy controls.\nTwelve messenger RNA biomarkers were discovered and validated. The logistic regression model with the combination of 4 messenger RNA biomarkers (KRAS, MBD3L2, ACRV1, and DPM1) could differentiate pancreatic cancer patients from noncancer subjects (chronic pancreatitis and healthy control), yielding a receiver operating characteristic plot, area under the curve value of 0.971 with 90.0% sensitivity and 95.0% specificity.\nThe salivary biomarkers possess discriminatory power for the detection of resectable pancreatic cancer, with high specificity and sensitivity. This report provides the proof of concept of salivary biomarkers for the noninvasive detection of a systemic cancer and paves the way for prediction model validation study followed by pivotal clinical validation.\nCopyright 2010 AGA Institute. Published by Elsevier Inc. All rights reserved.\n','2010-03-02',0,'PCA,Tumour purity,Gene expression,Correlations'),(337160,'20209107','Masitinib combined with standard gemcitabine chemotherapy: in vitro and in vivo studies in human pancreatic tumour cell lines and ectopic mouse model.','Humbert M,Castéran N,Letard S,Hanssens K,Iovanna J,Finetti P,Bertucci F,Bader T,Mansfield CD,Moussy A,Hermine O,Dubreuil P','PloS one','Tyrosine kinases are attractive targets for pancreatic cancer therapy because several are over-expressed, including PDGFRalpha/beta, FAK, Src and Lyn. A critical role of mast cells in the development of pancreatic cancer has also been reported. Masitinib is a tyrosine kinase inhibitor that selectively targets c-Kit, PDGFRalpha/beta, Lyn, and to a lesser extent the FAK pathway, without inhibiting kinases of known toxicities. Masitinib is particularly efficient in controlling the proliferation, differentiation and degranulation of mast cells. This study evaluates the therapeutic potential of masitinib in pancreatic cancer, as a single agent and in combination with gemcitabine.\nProof-of-concept studies were performed in vitro on human pancreatic tumour cell lines and then in vivo using a mouse model of human pancreatic cancer. Molecular mechanisms were investigated via gene expression profiling. Masitinib as a single agent had no significant antiproliferative activity while the masitinib/gemcitabine combination showed synergy in vitro on proliferation of gemcitabine-refractory cell lines Mia Paca2 and Panc1, and to a lesser extent in vivo on Mia Paca2 cell tumour growth. Specifically, masitinib at 10 microM strongly sensitised Mia Paca2 cells to gemcitabine (>400-fold reduction in IC(50)); and moderately sensitised Panc1 cells (10-fold reduction). Transcriptional analysis identified the Wnt/beta-catenin signalling pathway as down-regulated in the cell lines resensitised by the masitinib/gemcitabine combination.\nThese data establish proof-of-concept that masitinib can sensitise gemcitabine-refractory pancreatic cancer cell lines and warrant further in vivo investigation. Indeed, such an effect has been recently observed in a phase 2 clinical study of patients with pancreatic cancer who received a masitinib/gemcitabine combination.\n','2010-03-08',0,''),(337161,'20644708','A six-gene signature predicts survival of patients with localized pancreatic ductal adenocarcinoma.','Stratford JK,Bentrem DJ,Anderson JM,Fan C,Volmar KA,Marron JS,Routh ED,Caskey LS,Samuel JC,Der CJ,Thorne LB,Calvo BF,Kim HJ,Talamonti MS,Iacobuzio-Donahue CA,Hollingsworth MA,Perou CM,Yeh JJ','PLoS medicine','Pancreatic ductal adenocarcinoma (PDAC) remains a lethal disease. For patients with localized PDAC, surgery is the best option, but with a median survival of less than 2 years and a difficult and prolonged postoperative course for most, there is an urgent need to better identify patients who have the most aggressive disease.\nWe analyzed the gene expression profiles of primary tumors from patients with localized compared to metastatic disease and identified a six-gene signature associated with metastatic disease. We evaluated the prognostic potential of this signature in a training set of 34 patients with localized and resected PDAC and selected a cut-point associated with outcome using X-tile. We then applied this cut-point to an independent test set of 67 patients with localized and resected PDAC and found that our signature was independently predictive of survival and superior to established clinical prognostic factors such as grade, tumor size, and nodal status, with a hazard ratio of 4.1 (95% confidence interval [CI] 1.7-10.0). Patients defined to be high-risk patients by the six-gene signature had a 1-year survival rate of 55% compared to 91% in the low-risk group.\nOur six-gene signature may be used to better stage PDAC patients and assist in the difficult treatment decisions of surgery and to select patients whose tumor biology may benefit most from neoadjuvant therapy. The use of this six-gene signature should be investigated in prospective patient cohorts, and if confirmed, in future PDAC clinical trials, its potential as a biomarker should be investigated. Genes in this signature, or the pathways that they fall into, may represent new therapeutic targets. Please see later in the article for the Editors&apo; Summary.\n','2010-07-20',0,'PCA,Gene expression,Correlations,Survival analysis'),(337162,'20885998','Glycogene expression alterations associated with pancreatic cancer epithelial-mesenchymal transition in complementary model systems.','Maupin KA,Sinha A,Eugster E,Miller J,Ross J,Paulino V,Keshamouni VG,Tran N,Berens M,Webb C,Haab BB','PloS one','The ability to selectively detect and target cancer cells that have undergone an epithelial-mesenchymal transition (EMT) may lead to improved methods to treat cancers such as pancreatic cancer. The remodeling of cellular glycosylation previously has been associated with cell differentiation and may represent a valuable class of molecular targets for EMT.\nAs a first step toward investigating the nature of glycosylation alterations in EMT, we characterized the expression of glycan-related genes in three in-vitro model systems that each represented a complementary aspect of pancreatic cancer EMT. These models included: 1) TGFβ-induced EMT, which provided a look at the active transition between states; 2) a panel of 22 pancreatic cancer cell lines, which represented terminal differentiation states of either epithelial-like or mesenchymal-like; and 3) actively-migrating and stationary cells, which provided a look at the mechanism of migration. We analyzed expression data from a list of 587 genes involved in glycosylation (biosynthesis, sugar transport, glycan-binding, etc.) or EMT. Glycogenes were altered at a higher prevalence than all other genes in the first two models (p<0.05 and <0.005, respectively) but not in the migration model. Several functional themes were shared between the induced-EMT model and the cell line panel, including alterations to matrix components and proteoglycans, the sulfation of glycosaminoglycans; mannose receptor family members; initiation of O-glycosylation; and certain forms of sialylation. Protein-level changes were confirmed by Western blot for the mannose receptor MRC2 and the O-glycosylation enzyme GALNT3, and cell-surface sulfation changes were confirmed using Alcian Blue staining.\nAlterations to glycogenes are a major component of cancer EMT and are characterized by changes to matrix components, the sulfation of GAGs, mannose receptors, O-glycosylation, and specific sialylated structures. These results provide leads for targeting aggressive and drug resistant forms of pancreatic cancer cells.\n','2010-10-04',0,'PCA,Tumour purity,Gene expression,Correlations'),(337163,'21460848','Subtypes of pancreatic ductal adenocarcinoma and their differing responses to therapy.','Collisson EA,Sadanandam A,Olson P,Gibb WJ,Truitt M,Gu S,Cooc J,Weinkle J,Kim GE,Jakkula L,Feiler HS,Ko AH,Olshen AB,Danenberg KL,Tempero MA,Spellman PT,Hanahan D,Gray JW','Nature medicine','Pancreatic ductal adenocarcinoma (PDA) is a lethal disease. Overall survival is typically 6 months from diagnosis. Numerous phase 3 trials of agents effective in other malignancies have failed to benefit unselected PDA populations, although patients do occasionally respond. Studies in other solid tumors have shown that heterogeneity in response is determined, in part, by molecular differences between tumors. Furthermore, treatment outcomes are improved by targeting drugs to tumor subtypes in which they are selectively effective, with breast and lung cancers providing recent examples. Identification of PDA molecular subtypes has been frustrated by a paucity of tumor specimens available for study. We have overcome this problem by combined analysis of transcriptional profiles of primary PDA samples from several studies, along with human and mouse PDA cell lines. We define three PDA subtypes: classical, quasimesenchymal and exocrine-like, and we present evidence for clinical outcome and therapeutic response differences between them. We further define gene signatures for these subtypes that may have utility in stratifying patients for treatment and present preclinical model systems that may be used to identify new subtype specific therapies.\n','2011-04-08',0,'PCA,Gene expression,Correlations,Survival analysis'),(337164,'22261810','Integrative survival-based molecular profiling of human pancreatic cancer.','Donahue TR,Tran LM,Hill R,Li Y,Kovochich A,Calvopina JH,Patel SG,Wu N,Hindoyan A,Farrell JJ,Li X,Dawson DW,Wu H','Clinical cancer research : an official journal of the American Association for Cancer Research','To carry out an integrative profile of human pancreatic ductal adenocarcinoma (PDAC) to identify prognosis-significant genes and their related pathways.\nA concordant survival-based whole genome in silico array analysis of DNA copy number, and mRNA and miRNA expression in 25 early-stage PDAC was carried out. A novel composite score simultaneously integrated gene expression with regulatory mechanisms to identify the signature genes with the most levels of prognosis-significant evidence. The predominant signaling pathways were determined via a pathway-based approach. Independent patient cohorts (n = 148 and 42) were then used as in vitro validation of the array findings.\nThe composite score identified 171 genes in which expressions were able to define two prognosis subgroups (P = 3.8e-5). Eighty-eight percent (151 of 171) of the genes were regulated by prognosis-significant miRNAs. The phosphoinositide 3-kinase/AKT pathway and SRC signaling were densely populated by prognosis-significant genes and driven by genomic amplification of SRC and miRNA regulation of p85α and CBL. On tissue microarray validation (n = 148), p85α protein expression was associated with improved survival for all patients (P = 0.02), and activated P-SRC (Y418) was associated shorter survival for patients with low-grade histology tumors (P = 0.04). Interacting P-SRC and p85α revealed that they define two distinct PDAC patient subgroups (P = 0.0066). Furthering the importance of these pathways, CBL protein expression was associated with improved survival (P = 0.03) on a separate cohort (n = 42).\nThese pathways and related genes may represent putative clinical biomarkers and possible targets of individualized therapy in the distinct patient subgroups they define.\n','2012-03-05',0,''),(337165,'22273699','Gene expression changes associated with the progression of intraductal papillary mucinous neoplasms.','Jury RP,Thibodeau BJ,Fortier LE,Geddes TJ,Ahmed S,Pruetz BL,Farinola MA,Wilson GD','Pancreas','The diagnosis of high-grade intraductal papillary mucinous neoplasm (IPMN) is difficult to distinguish from low-grade IPMN. The aim of this study was to identify potential markers for the discrimination of high-grade and invasive (HgInv) IPMN from low- and moderate-grade dysplasia IPMN.\nLaser capture microdissection was used to isolate distinct foci of low-grade, moderate-grade, high-grade, and invasive IPMN from paraffin-embedded archival tissue from 14 patients who underwent resection for IPMN. Most samples included multiple grades in the same specimen. Affymetrix Human Exon microarrays were used to compare low- and moderate-grade dysplasia IPMN with HgInv IPMN.\nSixty-two genes were identified as showing significant changes in expression (P ≤ 0.05 and a 2-fold cutoff), including up-regulation of 41 in HgInv IPMN. Changes in gene expression are associated with biological processes related to malignant behavior including cell motion, cell proliferation, response to hypoxia, and epithelial-to-mesenchymal transition. In addition, altered signaling in several transforming growth factor β-related pathways was exhibited in the progression of IPMN to malignancy.\nThis study identifies a set of genes associated with the progression of IPMN to malignancy. These genes are potential markers that could be used to identify IPMN requiring surgical resection.\n','2012-04-17',0,''),(337166,'22363658','DPEP1 inhibits tumor cell invasiveness, enhances chemosensitivity and predicts clinical outcome in pancreatic ductal adenocarcinoma.','Zhang G,Schetter A,He P,Funamizu N,Gaedcke J,Ghadimi BM,Ried T,Hassan R,Yfantis HG,Lee DH,Lacy C,Maitra A,Hanna N,Alexander HR,Hussain SP','PloS one','Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancers worldwide. To identify biologically relevant genes with prognostic and therapeutic significance in PDAC, we first performed the microarray gene-expression profiling in 45 matching pairs of tumor and adjacent non-tumor tissues from resected PDAC cases. We identified 36 genes that were associated with patient outcome and also differentially expressed in tumors as compared with adjacent non-tumor tissues in microarray analysis. Further evaluation in an independent validation cohort (N = 27) confirmed that DPEP1 (dipeptidase 1) expression was decreased (T:N ratio ∼0.1, P<0.01) in tumors as compared with non-tumor tissues. DPEP1 gene expression was negatively correlated with histological grade (Spearman correlation coefficient = -0.35, P = 0.004). Lower expression of DPEP1 in tumors was associated with poor survival (Kaplan Meier log rank) in both test cohort (P = 0.035) and validation cohort (P = 0.016). DPEP1 expression was independently associated with cancer-specific mortality when adjusted for tumor stage and resection margin status in both univariate (hazard ratio = 0.43, 95%CI = 0.24-0.76, P = 0.004) and multivariate analyses (hazard ratio = 0.51, 95%CI = 0.27-0.94, P = 0.032). We further demonstrated that overexpression of DPEP1 suppressed tumor cells invasiveness and increased sensitivity to chemotherapeutic agent Gemcitabine. Our data also showed that growth factor EGF treatment decreased DPEP1 expression and MEK1/2 inhibitor AZD6244 increased DPEP1 expression in vitro, indicating a potential mechanism for DPEP1 gene regulation. Therefore, we provide evidence that DPEP1 plays a role in pancreatic cancer aggressiveness and predicts outcome in patients with resected PDAC. In view of these findings, we propose that DPEP1 may be a candidate target in PDAC for designing improved treatments.\n','2012-02-24',0,'PCA,Gene expression,Correlations,Survival analysis'),(337167,'22763454','RNA sequencing of pancreatic circulating tumour cells implicates WNT signalling in metastasis.','Yu M,Ting DT,Stott SL,Wittner BS,Ozsolak F,Paul S,Ciciliano JC,Smas ME,Winokur D,Gilman AJ,Ulman MJ,Xega K,Contino G,Alagesan B,Brannigan BW,Milos PM,Ryan DP,Sequist LV,Bardeesy N,Ramaswamy S,Toner M,Maheswaran S,Haber DA','Nature','Circulating tumour cells (CTCs) shed into blood from primary cancers include putative precursors that initiate distal metastases. Although these cells are extraordinarily rare, they may identify cellular pathways contributing to the blood-borne dissemination of cancer. Here, we adapted a microfluidic device for efficient capture of CTCs from an endogenous mouse pancreatic cancer model and subjected CTCs to single-molecule RNA sequencing, identifying Wnt2 as a candidate gene enriched in CTCs. Expression of WNT2 in pancreatic cancer cells suppresses anoikis, enhances anchorage-independent sphere formation, and increases metastatic propensity in vivo. This effect is correlated with fibronectin upregulation and suppressed by inhibition of MAP3K7 (also known as TAK1) kinase. In humans, formation of non-adherent tumour spheres by pancreatic cancer cells is associated with upregulation of multiple WNT genes, and pancreatic CTCs revealed enrichment for WNT signalling in 5 out of 11 cases. Thus, molecular analysis of CTCs may identify candidate therapeutic targets to prevent the distal spread of cancer.\n','2012-07-27',0,''),(337168,'23007696','S100P is a metastasis-associated gene that facilitates transendothelial migration of pancreatic cancer cells.','Barry S,Chelala C,Lines K,Sunamura M,Wang A,Marelli-Berg FM,Brennan C,Lemoine NR,Crnogorac-Jurcevic T','Clinical & experimental metastasis','Pancreatic ductal adenocarcinoma (PDAC) is the 5th most common cause of cancer death in the UK and the 4th in the US. The vast majority of deaths following pancreatic cancer are due to metastatic spread, hence understanding the metastatic process is vital for identification of critically needed novel therapeutic targets. An enriched set of 33 genes differentially expressed in common between primary PDAC and liver metastases, when compared to normal tissues, was obtained through global gene expression profiling. This metastasis-associated gene set comprises transcripts from both cancer (S100P, S100A6, AGR2, etc.) and adjacent stroma (collagens type I, III, and V, etc.), thus reinforcing the concept of a continuous crosstalk between the two compartments in both primary tumours and their metastases. The expression of S100P, SFN, VCAN and collagens was further validated in additional primary PDACs and matched liver metastatic lesions, while the functional significance of one of the most highly expressed genes, S100P, was studied in more detail. We show that this protein increases the transendothelial migration of PDAC cancer cells in vitro, which was also confirmed in vivo experiments using a zebrafish embryo model. Thus S100P facilitates cancer cell intravasation/extravasation, critical steps in the hematogenous dissemination of pancreatic cancer cells.\n','2013-02-13',0,'PCA,Tumour purity,Gene expression,Correlations'),(337169,'23372777','Molecular analysis of precursor lesions in familial pancreatic cancer.','Crnogorac-Jurcevic T,Chelala C,Barry S,Harada T,Bhakta V,Lattimore S,Jurcevic S,Bronner M,Lemoine NR,Brentnall TA','PloS one','With less than a 5% survival rate pancreatic adenocarcinoma (PDAC) is almost uniformly lethal. In order to make a significant impact on survival of patients with this malignancy, it is necessary to diagnose the disease early, when curative surgery is still possible. Detailed knowledge of the natural history of the disease and molecular events leading to its progression is therefore critical.\nWe have analysed the precursor lesions, PanINs, from prophylactic pancreatectomy specimens of patients from four different kindreds with high risk of familial pancreatic cancer who were treated for histologically proven PanIN-2/3. Thus, the material was procured before pancreatic cancer has developed, rather than from PanINs in a tissue field that already contains cancer. Genome-wide transcriptional profiling using such unique specimens was performed. Bulk frozen sections displaying the most extensive but not microdissected PanIN-2/3 lesions were used in order to obtain the holistic view of both the precursor lesions and their microenvironment. A panel of 76 commonly dysregulated genes that underlie neoplastic progression from normal pancreas to PanINs and PDAC were identified. In addition to shared genes some differences between the PanINs of individual families as well as between the PanINs and PDACs were also seen. This was particularly pronounced in the stromal and immune responses.\nOur comprehensive analysis of precursor lesions without the invasive component provides the definitive molecular proof that PanIN lesions beget cancer from a molecular standpoint. We demonstrate the need for accumulation of transcriptomic changes during the progression of PanIN to PDAC, both in the epithelium and in the surrounding stroma. An identified 76-gene signature of PDAC progression presents a rich candidate pool for the development of early diagnostic and/or surveillance markers as well as potential novel preventive/therapeutic targets for both familial and sporadic pancreatic adenocarcinoma.\n','2013-02-01',0,'PCA,Tumour purity,Gene expression,Correlations'),(337170,'24204737','Clinical, molecular and genetic validation of a murine orthotopic xenograft model of pancreatic adenocarcinoma using fresh human specimens.','Walters DM,Stokes JB,Adair SJ,Stelow EB,Borgman CA,Lowrey BT,Xin W,Blais EM,Lee JK,Papin JA,Parsons JT,Bauer TW','PloS one','Relevant preclinical models that recapitulate the key features of human pancreatic ductal adenocarcinoma (PDAC) are needed in order to provide biologically tractable models to probe disease progression and therapeutic responses and ultimately improve patient outcomes for this disease. Here, we describe the establishment and clinical, pathological, molecular and genetic validation of a murine, orthotopic xenograft model of PDAC.\nHuman PDACs were resected and orthotopically implanted and propagated in immunocompromised mice. Patient survival was correlated with xenograft growth and metastatic rate in mice. Human and mouse tumor pathology were compared. Tumors were analyzed for genetic mutations, gene expression, receptor tyrosine kinase activation, and cytokine expression.\nFifteen human PDACs were propagated orthotopically in mice. Xenograft-bearing mice developed peritoneal and liver metastases. Time to tumor growth and metastatic efficiency in mice each correlated with patient survival. Tumor architecture, nuclear grade and stromal content were similar in patient and xenografted tumors. Propagated tumors closely exhibited the genetic and molecular features known to characterize pancreatic cancer (e.g. high rate of KRAS, P53, SMAD4 mutation and EGFR activation). The correlation coefficient of gene expression between patient tumors and xenografts propagated through multiple generations was 93 to 99%. Analysis of gene expression demonstrated distinct differences between xenografts from fresh patient tumors versus commercially available PDAC cell lines.\nThe orthotopic xenograft model derived from fresh human PDACs closely recapitulates the clinical, pathologic, genetic and molecular aspects of human disease. This model has resulted in the identification of rational therapeutic strategies to be tested in clinical trials and will permit additional therapeutic approaches and identification of biomarkers of response to therapy.\n','2013-11-08',0,''),(337171,'24498386','A GNAS mutation found in pancreatic intraductal papillary mucinous neoplasms induces drastic alterations of gene expression profiles with upregulation of mucin genes.','Komatsu H,Tanji E,Sakata N,Aoki T,Motoi F,Naitoh T,Katayose Y,Egawa S,Unno M,Furukawa T','PloS one','GNAS, a gene encoding G protein stimulating α subunit, is frequently mutated in intraductal papillary mucinous neoplasms (IPMNs), which are indolent and slow-growing pancreatic tumors that secrete abundant mucin. The GNAS mutation is not observed in conventional ductal adenocarcinomas of the pancreas. To determine the functional significance of the GNAS mutation in pancreatic ductal lineage cells, we examined in vitro phenotypes of cells of pancreatic ductal lineage, HPDE, PK-8, PCI-35, and MIA PaCa-2, with exogenous expression of either wild-type or mutated (R201H) GNAS. We found that exogenous GNAS upregulated intracellular cyclic adenine monophosphate (cAMP), particularly in mutated GNAS transfectants, and upregulated expression of MUC2 and MUC5AC in HPDE and PK-8 cells. By contrast, exogenous GNAS inhibited expression of mucin genes in PCI-35 and MIA PaCa-2 cells, despite upregulation of cAMP. We examined global gene expression profiles of some of the cells transfected with exogenous mutated GNAS (PK-8, PCI-35, and MIA PaCa-2), and found that PK-8 cells exhibited drastic alterations of the gene expression profile, which contrasted with modest alterations in PCI-35 and MIA PaCa-2 cells. To identify a cause of these different effects of exogenous mutated GNAS on phenotypes of the cells, we examined effects of interactions of the signaling pathways of G protein-coupled receptor (GPCR), mitogen-activated protein kinase (MAPK), and phosphatidylinositol 3-kinase (PI3K) on expression of mucin genes. The MAPK and PI3K pathways significantly influenced the expression of mucin genes. Exogenous GNAS did not promote cell growth but suppressed it in some of the cells. In conclusion, mutated GNAS found in IPMNs may extensively alter gene expression profiles, including expression of mucin genes, through the interaction with MAPK and PI3K pathways in pancreatic ductal cells; these changes may determine the characteristic phenotype of IPMN. PK-8 cells expressing exogenous mutated GNAS may be an ideal in vitro model of IPMN.\n','2014-02-05',0,''),(337172,'24739241','Transcriptional dissection of pancreatic tumors engrafted in mice.','Martinez-Garcia R,Juan D,Rausell A,Muñoz M,Baños N,Menéndez C,Lopez-Casas PP,Rico D,Valencia A,Hidalgo M','Genome medicine','Engraftment of primary pancreas ductal adenocarcinomas (PDAC) in mice to generate patient-derived xenograft (PDX) models is a promising platform for biological and therapeutic studies in this disease. However, these models are still incompletely characterized. Here, we measured the impact of the murine tumor environment on the gene expression of the engrafted human tumoral cells.\nWe have analyzed gene expression profiles from 35 new PDX models and compared them with previously published microarray data of 18 PDX models, 53 primary tumors and 41 cell lines from PDAC. The results obtained in the PDAC system were further compared with public available microarray data from 42 PDX models, 108 primary tumors and 32 cell lines from hepatocellular carcinoma (HCC). We developed a robust analysis protocol to explore the gene expression space. In addition, we completed the analysis with a functional characterization of PDX models, including if changes were caused by murine environment or by serial passing.\nOur results showed that PDX models derived from PDAC, or HCC, were clearly different to the cell lines derived from the same cancer tissues. Indeed, PDAC- and HCC-derived cell lines are indistinguishable from each other based on their gene expression profiles. In contrast, the transcriptomes of PDAC and HCC PDX models can be separated into two different groups that share some partial similarity with their corresponding original primary tumors. Our results point to the lack of human stromal involvement in PDXs as a major factor contributing to their differences from the original primary tumors. The main functional differences between pancreatic PDX models and human PDAC are the lower expression of genes involved in pathways related to extracellular matrix and hemostasis and the up- regulation of cell cycle genes. Importantly, most of these differences are detected in the first passages after the tumor engraftment.\nOur results suggest that PDX models of PDAC and HCC retain, to some extent, a gene expression memory of the original primary tumors, while this pattern is not detected in conventional cancer cell lines. Expression changes in PDXs are mainly related to pathways reflecting the lack of human infiltrating cells and the adaptation to a new environment. We also provide evidence of the stability of gene expression patterns over subsequent passages, indicating early phases of the adaptation process.\n','2014-08-12',0,'PCA,Tumour purity,Gene expression,Correlations'),(337173,'25242334','Single-cell RNA sequencing identifies extracellular matrix gene expression by pancreatic circulating tumor cells.','Ting DT,Wittner BS,Ligorio M,Vincent Jordan N,Shah AM,Miyamoto DT,Aceto N,Bersani F,Brannigan BW,Xega K,Ciciliano JC,Zhu H,MacKenzie OC,Trautwein J,Arora KS,Shahid M,Ellis HL,Qu N,Bardeesy N,Rivera MN,Deshpande V,Ferrone CR,Kapur R,Ramaswamy S,Shioda T,Toner M,Maheswaran S,Haber DA','Cell reports','Circulating tumor cells (CTCs) are shed from primary tumors into the bloodstream, mediating the hematogenous spread of cancer to distant organs. To define their composition, we compared genome-wide expression profiles of CTCs with matched primary tumors in a mouse model of pancreatic cancer, isolating individual CTCs using epitope-independent microfluidic capture, followed by single-cell RNA sequencing. CTCs clustered separately from primary tumors and tumor-derived cell lines, showing low-proliferative signatures, enrichment for the stem-cell-associated gene Aldh1a2, biphenotypic expression of epithelial and mesenchymal markers, and expression of Igfbp5, a gene transcript enriched at the epithelial-stromal interface. Mouse as well as human pancreatic CTCs exhibit a very high expression of stromal-derived extracellular matrix (ECM) proteins, including SPARC, whose knockdown in cancer cells suppresses cell migration and invasiveness. The aberrant expression by CTCs of stromal ECM genes points to their contribution of microenvironmental signals for the spread of cancer to distant organs.\nCopyright © 2014 The Authors. Published by Elsevier Inc. All rights reserved.\n','2014-09-27',0,''),(337174,'25259922','Vitamin D receptor-mediated stromal reprogramming suppresses pancreatitis and enhances pancreatic cancer therapy.','Sherman MH,Yu RT,Engle DD,Ding N,Atkins AR,Tiriac H,Collisson EA,Connor F,Van Dyke T,Kozlov S,Martin P,Tseng TW,Dawson DW,Donahue TR,Masamune A,Shimosegawa T,Apte MV,Wilson JS,Ng B,Lau SL,Gunton JE,Wahl GM,Hunter T,Drebin JA,O&apo;Dwyer PJ,Liddle C,Tuveson DA,Downes M,Evans RM','Cell','The poor clinical outcome in pancreatic ductal adenocarcinoma (PDA) is attributed to intrinsic chemoresistance and a growth-permissive tumor microenvironment. Conversion of quiescent to activated pancreatic stellate cells (PSCs) drives the severe stromal reaction that characterizes PDA. Here, we reveal that the vitamin D receptor (VDR) is expressed in stroma from human pancreatic tumors and that treatment with the VDR ligand calcipotriol markedly reduced markers of inflammation and fibrosis in pancreatitis and human tumor stroma. We show that VDR acts as a master transcriptional regulator of PSCs to reprise the quiescent state, resulting in induced stromal remodeling, increased intratumoral gemcitabine, reduced tumor volume, and a 57% increase in survival compared to chemotherapy alone. This work describes a molecular strategy through which transcriptional reprogramming of tumor stroma enables chemotherapeutic response and suggests vitamin D priming as an adjunct in PDA therapy. PAPERFLICK:\nCopyright © 2014 Elsevier Inc. All rights reserved.\n','2014-09-27',0,''),(337175,'25336113','SOX9 regulates ERBB signalling in pancreatic cancer development.','Grimont A,Pinho AV,Cowley MJ,Augereau C,Mawson A,Giry-Laterrière M,Van den Steen G,Waddell N,Pajic M,Sempoux C,Wu J,Grimmond SM,Biankin AV,Lemaigre FP,Rooman I,Jacquemin P','Gut','The transcription factor SOX9 was recently shown to stimulate ductal gene expression in pancreatic acinar-to-ductal metaplasia and to accelerate development of premalignant lesions preceding pancreatic ductal adenocarcinoma (PDAC). Here, we investigate how SOX9 operates in pancreatic tumourigenesis.\nWe analysed genomic and transcriptomic data from surgically resected PDAC and extended the expression analysis to xenografts from PDAC samples and to PDAC cell lines. SOX9 expression was manipulated in human cell lines and mouse models developing PDAC.\nWe found genetic aberrations in the SOX9 gene in about 15% of patient tumours. Most PDAC samples strongly express SOX9 protein, and SOX9 levels are higher in classical PDAC. This tumour subtype is associated with better patient outcome, and cell lines of this subtype respond to therapy targeting epidermal growth factor receptor (EGFR/ERBB1) signalling, a pathway essential for pancreatic tumourigenesis. In human PDAC, high expression of SOX9 correlates with expression of genes belonging to the ERBB pathway. In particular, ERBB2 expression in PDAC cell lines is stimulated by SOX9. Inactivating Sox9 expression in mice confirmed its role in PDAC initiation; it demonstrated that Sox9 stimulates expression of several members of the ERBB pathway and is required for ERBB signalling activity.\nBy integrating data from patient samples and mouse models, we found that SOX9 regulates the ERBB pathway throughout pancreatic tumourigenesis. Our work opens perspectives for therapy targeting tumourigenic mechanisms.\nPublished by the BMJ Publishing Group Limited. For permission to use (where not already granted under a licence) please go to http://group.bmj.com/group/rights-licensing/permissions.\n','2015-10-09',0,'PCA,Gene expression,Correlations,Survival analysis'),(337176,'25579086','Molecular signatures of mRNAs and miRNAs as prognostic biomarkers in pancreatobiliary and intestinal types of periampullary adenocarcinomas.','Sandhu V,Bowitz Lothe IM,Labori KJ,Lingjærde OC,Buanes T,Dalsgaard AM,Skrede ML,Hamfjord J,Haaland T,Eide TJ,Børresen-Dale AL,Ikdahl T,Kure EH','Molecular oncology','Periampullary adenocarcinomas include four anatomical sites of origin (the pancreatic duct, bile duct, ampulla and duodenum) and most of them fall into two histological subgroups (pancreatobiliary and intestinal). Determining the exact origin of the tumor is sometimes difficult, due to overlapping histopathological characteristics. The prognosis depends on the histological subtype, as well as on the anatomical site of origin, the former being the more important. The molecular basis for these differences in prognosis is poorly understood. Whole-genome analyses were used to investigate the association between molecular tumor profiles, pathogenesis and prognosis. A total of 85 periampullary adenocarcinomas were characterized by mRNA and miRNA expressions profiling. Molecular profiles of the tumors from the different anatomical sites of origin as well as of the different histological subtypes were compared. Differentially expressed mRNAs and miRNAs between the two histopathological subtypes were linked to specific molecular pathways. Six miRNA families were downregulated and four were upregulated in the pancreatobiliary type as compared to the intestinal type (P < 0.05). miRNAs and mRNAs associated with improved overall and recurrence free survival for the two histopathological subtypes were identified. For the pancreatobiliary type the genes ATM, PTEN, RB1 and the miRNAs miR-592 and miR-497, and for the intestinal type the genes PDPK1, PIK3R2, G6PC and the miRNAs miR-127-3p, miR-377* were linked to enriched pathways and identified as prognostic markers. The molecular signatures identified may in the future guide the clinicians in the therapeutic decision making to an individualized treatment, if confirmed in other larger datasets.\nCopyright © 2014 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.\n','2015-04-06',0,''),(337177,'25601205','Endothelial cells control pancreatic cell fate at defined stages through EGFL7 signaling.','Kao DI,Lacko LA,Ding BS,Huang C,Phung K,Gu G,Rafii S,Stuhlmann H,Chen S','Stem cell reports','Although endothelial cells have been shown to affect mouse pancreatic development, their precise function in human development remains unclear. Using a coculture system containing human embryonic stem cell (hESC)-derived progenitors and endothelial cells, we found that endothelial cells play a stage-dependent role in pancreatic development, in which they maintain pancreatic progenitor (PP) self-renewal and impair further differentiation into hormone-expressing cells. The mechanistic studies suggest that the endothelial cells act through the secretion of EGFL7. Consistently, endothelial overexpression of EGFL7 in vivo using a transgenic mouse model resulted in an increase of PP proliferation rate and a decrease of differentiation toward endocrine cells. These studies not only identified the role of EGFL7 as the molecular handle involved in the crosstalk between endothelium and pancreatic epithelium, but also provide a paradigm for using hESC stepwise differentiation to dissect the stage-dependent roles of signals controlling organogenesis.\nCopyright © 2015 The Authors. Published by Elsevier Inc. All rights reserved.\n','2015-02-14',0,''),(337178,'25607660','A genome-wide investigation of microRNA expression identifies biologically-meaningful microRNAs that distinguish between high-risk and low-risk intraductal papillary mucinous neoplasms of the pancreas.','Permuth-Wey J,Chen YA,Fisher K,McCarthy S,Qu X,Lloyd MC,Kasprzak A,Fournier M,Williams VL,Ghia KM,Yoder SJ,Hall L,Georgeades C,Olaoye F,Husain K,Springett GM,Chen DT,Yeatman T,Centeno BA,Klapman J,Coppola D,Malafa M','PloS one','Intraductal papillary mucinous neoplasms (IPMNs) are pancreatic ductal adenocarcinoma (PDAC) precursors. Differentiating between high-risk IPMNs that warrant surgical resection and low-risk IPMNs that can be monitored is a significant clinical problem, and we sought to discover a panel of mi(cro)RNAs that accurately classify IPMN risk status.\nIn a discovery phase, genome-wide miRNA expression profiling was performed on 28 surgically-resected, pathologically-confirmed IPMNs (19 high-risk, 9 low-risk) using Taqman MicroRNA Arrays. A validation phase was performed in 21 independent IPMNs (13 high-risk, 8 low-risk). We also explored associations between miRNA expression level and various clinical and pathological factors and examined genes and pathways regulated by the identified miRNAs by integrating data from bioinformatic analyses and microarray analysis of miRNA gene targets. Six miRNAs (miR-100, miR-99b, miR-99a, miR-342-3p, miR-126, miR-130a) were down-regulated in high-risk versus low-risk IPMNs and distinguished between groups (P<10-3, area underneath the curve (AUC) = 87%). The same trend was observed in the validation phase (AUC = 74%). Low miR-99b expression was associated with main pancreatic duct involvement (P = 0.021), and serum albumin levels were positively correlated with miR-99a (r = 0.52, P = 0.004) and miR-100 expression (r = 0.49, P = 0.008). Literature, validated miRNA:target gene interactions, and pathway enrichment analysis supported the candidate miRNAs as tumor suppressors and regulators of PDAC development. Microarray analysis revealed that oncogenic targets of miR-130a (ATG2B, MEOX2), miR-342-3p (DNMT1), and miR-126 (IRS-1) were up-regulated in high- versus low-risk IPMNs (P<0.10).\nThis pilot study highlights miRNAs that may aid in preoperative risk stratification of IPMNs and provides novel insights into miRNA-mediated progression to pancreatic malignancy. The miRNAs identified here and in other recent investigations warrant evaluation in biofluids in a well-powered prospective cohort of individuals newly-diagnosed with IPMNs and other pancreatic cysts and those at increased genetic risk for these lesions.\n','2015-01-22',0,''),(337179,'26168401','Transcriptional control of autophagy-lysosome function drives pancreatic cancer metabolism.','Perera RM,Stoykova S,Nicolay BN,Ross KN,Fitamant J,Boukhali M,Lengrand J,Deshpande V,Selig MK,Ferrone CR,Settleman J,Stephanopoulos G,Dyson NJ,Zoncu R,Ramaswamy S,Haas W,Bardeesy N','Nature','Activation of cellular stress response pathways to maintain metabolic homeostasis is emerging as a critical growth and survival mechanism in many cancers. The pathogenesis of pancreatic ductal adenocarcinoma (PDA) requires high levels of autophagy, a conserved self-degradative process. However, the regulatory circuits that activate autophagy and reprogram PDA cell metabolism are unknown. Here we show that autophagy induction in PDA occurs as part of a broader transcriptional program that coordinates activation of lysosome biogenesis and function, and nutrient scavenging, mediated by the MiT/TFE family of transcription factors. In human PDA cells, the MiT/TFE proteins--MITF, TFE3 and TFEB--are decoupled from regulatory mechanisms that control their cytoplasmic retention. Increased nuclear import in turn drives the expression of a coherent network of genes that induce high levels of lysosomal catabolic function essential for PDA growth. Unbiased global metabolite profiling reveals that MiT/TFE-dependent autophagy-lysosome activation is specifically required to maintain intracellular amino acid pools. These results identify the MiT/TFE proteins as master regulators of metabolic reprogramming in pancreatic cancer and demonstrate that transcriptional activation of clearance pathways converging on the lysosome is a novel hallmark of aggressive malignancy.\n','2015-08-20',0,''),(337180,'26191083','Altering cancer transcriptomes using epigenomic inhibitors.','Gaddis M,Gerrard D,Frietze S,Farnham PJ','Epigenetics & chromatin','Due to the hyper-activation of WNT signaling in a variety of cancer types, there has been a strong drive to develop pathway-specific inhibitors with the eventual goal of providing a chemotherapeutic antagonist of WNT signaling to cancer patients. A new category of drugs, called epigenetic inhibitors, are being developed that hold high promise for inhibition of the WNT pathway. The canonical WNT signaling pathway initiates when WNT ligands bind to receptors, causing the nuclear localization of the co-activator β-catenin (CTNNB1), which leads to an association of β-catenin with a member of the TCF transcription factor family at regulatory regions of WNT-responsive genes. The TCF/β-catenin complex then recruits CBP (CREBBP) or p300 (EP300), leading to histone acetylation and gene activation. A current model in the field is that CBP-driven expression of WNT target genes supports proliferation whereas p300-driven expression of WNT target genes supports differentiation. The small molecule inhibitor ICG-001 binds to CBP, but not to p300, and competitively inhibits the interaction of CBP with β-catenin. Upon treatment of cancer cells, this should reduce expression of CBP-regulated transcription, leading to reduced tumorigenicity and enhanced differentiation.\nWe have compared the genome-wide effects on the transcriptome after treatment with ICG-001 (the specific CBP inhibitor) versus C646, a compound that competes with acetyl-coA for the Lys-coA binding pocket of both CBP and p300. We found that both drugs cause large-scale changes in the transcriptome of HCT116 colon cancer cells and PANC1 pancreatic cancer cells and reverse some tumor-specific changes in gene expression. Interestingly, although the epigenetic inhibitors affect cell cycle pathways in both the colon and pancreatic cancer cell lines, the WNT signaling pathway was affected only in the colon cancer cells. Notably, WNT target genes were similarly downregulated after treatment of HCT116 with C646 as with ICG-001.\nOur results suggest that treatment with a general HAT inhibitor causes similar effects on the transcriptome as does treatment with a CBP-specific inhibitor and that epigenetic inhibition affects the WNT pathway in HCT116 cells and the cholesterol biosynthesis pathway in PANC1 cells.\n','2015-07-20',0,''),(337181,'26343385','Virtual microdissection identifies distinct tumor- and stroma-specific subtypes of pancreatic ductal adenocarcinoma.','Moffitt RA,Marayati R,Flate EL,Volmar KE,Loeza SG,Hoadley KA,Rashid NU,Williams LA,Eaton SC,Chung AH,Smyla JK,Anderson JM,Kim HJ,Bentrem DJ,Talamonti MS,Iacobuzio-Donahue CA,Hollingsworth MA,Yeh JJ','Nature genetics','Pancreatic ductal adenocarcinoma (PDAC) remains a lethal disease with a 5-year survival rate of 4%. A key hallmark of PDAC is extensive stromal involvement, which makes capturing precise tumor-specific molecular information difficult. Here we have overcome this problem by applying blind source separation to a diverse collection of PDAC gene expression microarray data, including data from primary tumor, metastatic and normal samples. By digitally separating tumor, stromal and normal gene expression, we have identified and validated two tumor subtypes, including a &apo;basal-like&apo; subtype that has worse outcome and is molecularly similar to basal tumors in bladder and breast cancers. Furthermore, we define &apo;normal&apo; and &apo;activated&apo; stromal subtypes, which are independently prognostic. Our results provide new insights into the molecular composition of PDAC, which may be used to tailor therapies or provide decision support in a clinical setting where the choice and timing of therapies are critical.\n','2015-09-30',0,'PCA,Gene expression,Correlations,Survival analysis'),(337182,'26769127','Dissection of transcriptional and cis-regulatory control of differentiation in human pancreatic cancer.','Diaferia GR,Balestrieri C,Prosperini E,Nicoli P,Spaggiari P,Zerbi A,Natoli G','The EMBO journal','The histological grade of carcinomas describes the ability of tumor cells to organize in differentiated epithelial structures and has prognostic and therapeutic impact. Here, we show that differential usage of the genomic repertoire of transcriptional enhancers leads to grade-specific gene expression programs in human pancreatic ductal adenocarcinoma (PDAC). By integrating gene expression profiling, epigenomic footprinting, and loss-of-function experiments in PDAC cell lines of different grade, we identified the repertoires of enhancers specific to high- and low-grade PDACs and the cognate set of transcription factors acting to maintain their activity. Among the candidate regulators of PDAC differentiation, KLF5 was selectively expressed in pre-neoplastic lesions and low-grade primary PDACs and cell lines, where it maintained the acetylation of grade-specific enhancers, the expression of epithelial genes such as keratins and mucins, and the ability to organize glandular epithelia in xenografts. The identification of the transcription factors controlling differentiation in PDACs will help clarify the molecular bases of its heterogeneity and progression.\n© 2016 The Authors.\n','2016-03-17',0,''),(337183,'27281208','Image-based detection and targeting of therapy resistance in pancreatic adenocarcinoma.','Fox RG,Lytle NK,Jaquish DV,Park FD,Ito T,Bajaj J,Koechlein CS,Zimdahl B,Yano M,Kopp JL,Kritzik M,Sicklick JK,Sander M,Grandgenett PM,Hollingsworth MA,Shibata S,Pizzo D,Valasek MA,Sasik R,Scadeng M,Okano H,Kim Y,MacLeod AR,Lowy AM,Reya T','Nature','Pancreatic intraepithelial neoplasia is a pre-malignant lesion that can progress to pancreatic ductal adenocarcinoma, a highly lethal malignancy marked by its late stage at clinical presentation and profound drug resistance. The genomic alterations that commonly occur in pancreatic cancer include activation of KRAS2 and inactivation of p53 and SMAD4 (refs 2-4). So far, however, it has been challenging to target these pathways therapeutically; thus the search for other key mediators of pancreatic cancer growth remains an important endeavour. Here we show that the stem cell determinant Musashi (Msi) is a critical element of pancreatic cancer progression both in genetic models and in patient-derived xenografts. Specifically, we developed Msi reporter mice that allowed image-based tracking of stem cell signals within cancers, revealing that Msi expression rises as pancreatic intraepithelial neoplasia progresses to adenocarcinoma, and that Msi-expressing cells are key drivers of pancreatic cancer: they preferentially harbour the capacity to propagate adenocarcinoma, are enriched in circulating tumour cells, and are markedly drug resistant. This population could be effectively targeted by deletion of either Msi1 or Msi2, which led to a striking defect in the progression of pancreatic intraepithelial neoplasia to adenocarcinoma and an improvement in overall survival. Msi inhibition also blocked the growth of primary patient-derived tumours, suggesting that this signal is required for human disease. To define the translational potential of this work we developed antisense oligonucleotides against Msi; these showed reliable tumour penetration, uptake and target inhibition, and effectively blocked pancreatic cancer growth. Collectively, these studies highlight Msi reporters as a unique tool to identify therapy resistance, and define Msi signalling as a central regulator of pancreatic cancer.\n','2016-06-16',0,''),(337184,'27282075','RNA sequencing of pancreatic adenocarcinoma tumors yields novel expression patterns associated with long-term survival and reveals a role for ANGPTL4.','Kirby MK,Ramaker RC,Gertz J,Davis NS,Johnston BE,Oliver PG,Sexton KC,Greeno EW,Christein JD,Heslin MJ,Posey JA,Grizzle WE,Vickers SM,Buchsbaum DJ,Cooper SJ,Myers RM','Molecular oncology','Pancreatic adenocarcinoma patients have low survival rates due to late-stage diagnosis and high rates of cancer recurrence even after surgical resection. It is important to understand the molecular characteristics associated with survival differences in pancreatic adenocarcinoma tumors that may inform patient care.\nRNA sequencing was performed for 51 patient tumor tissues extracted from patients undergoing surgical resection, and expression was associated with overall survival time from diagnosis. Our analysis uncovered 323 transcripts whose expression correlates with survival time in our pancreatic patient cohort. This genomic signature was validated in an independent RNA-seq dataset of 68 additional patients from the International Cancer Genome Consortium. We demonstrate that this transcriptional profile is largely independent of markers of cellular division and present a 19-transcript predictive model built from a subset of the 323 transcripts that can distinguish patients with differing survival times across both the training and validation patient cohorts. We present evidence that a subset of the survival-associated transcripts is associated with resistance to gemcitabine treatment in vitro, and reveal that reduced expression of one of the survival-associated transcripts, Angiopoietin-like 4, impairs growth of a gemcitabine-resistant pancreatic cancer cell line.\nGene expression patterns in pancreatic adenocarcinoma tumors can distinguish patients with differing survival outcomes after undergoing surgical resection, and the survival difference could be associated with the intrinsic gemcitabine sensitivity of primary patient tumors. Thus, these transcriptional differences may impact patient care by distinguishing patients who would benefit from a non-gemcitabine based therapy.\nCopyright © 2016 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.\n','2016-09-17',0,'PCA,Gene expression,Correlations,Survival analysis'),(337185,'27345837','De Novo Prediction of Stem Cell Identity using Single-Cell Transcriptome Data.','Grün D,Muraro MJ,Boisset JC,Wiebrands K,Lyubimova A,Dharmadhikari G,van den Born M,van Es J,Jansen E,Clevers H,de Koning EJ,van Oudenaarden A','Cell stem cell','Adult mitotic tissues like the intestine, skin, and blood undergo constant turnover throughout the life of an organism. Knowing the identity of the stem cell is crucial to understanding tissue homeostasis and its aberrations upon disease. Here we present a computational method for the derivation of a lineage tree from single-cell transcriptome data. By exploiting the tree topology and the transcriptome composition, we establish StemID, an algorithm for identifying stem cells among all detectable cell types within a population. We demonstrate that StemID recovers two known adult stem cell populations, Lgr5+ cells in the small intestine and hematopoietic stem cells in the bone marrow. We apply StemID to predict candidate multipotent cell populations in the human pancreas, a tissue with largely uncharacterized turnover dynamics. We hope that StemID will accelerate the search for novel stem cells by providing concrete markers for biological follow-up and validation.\nCopyright © 2016 The Authors. Published by Elsevier Inc. All rights reserved.\n','2016-08-06',0,''),(337186,'27498862','Integrated Patient-Derived Models Delineate Individualized Therapeutic Vulnerabilities of Pancreatic Cancer.','Witkiewicz AK,Balaji U,Eslinger C,McMillan E,Conway W,Posner B,Mills GB,O&apo;Reilly EM,Knudsen ES','Cell reports','Pancreatic ductal adenocarcinoma (PDAC) harbors the worst prognosis of any common solid tumor, and multiple failed clinical trials indicate therapeutic recalcitrance. Here, we use exome sequencing of patient tumors and find multiple conserved genetic alterations. However, the majority of tumors exhibit no clearly defined therapeutic target. High-throughput drug screens using patient-derived cell lines found rare examples of sensitivity to monotherapy, with most models requiring combination therapy. Using PDX models, we confirmed the effectiveness and selectivity of the identified treatment responses. Out of more than 500 single and combination drug regimens tested, no single treatment was effective for the majority of PDAC tumors, and each case had unique sensitivity profiles that could not be predicted using genetic analyses. These data indicate a shortcoming of reliance on genetic analysis to predict efficacy of currently available agents against PDAC and suggest that sensitivity profiling of patient-derived models could inform personalized therapy design for PDAC.\nCopyright © 2016 The Author(s). Published by Elsevier Inc. All rights reserved.\n','2016-08-18',0,''),(337187,'28092686','Epigenomic reprogramming during pancreatic cancer progression links anabolic glucose metabolism to distant metastasis.','McDonald OG,Li X,Saunders T,Tryggvadottir R,Mentch SJ,Warmoes MO,Word AE,Carrer A,Salz TH,Natsume S,Stauffer KM,Makohon-Moore A,Zhong Y,Wu H,Wellen KE,Locasale JW,Iacobuzio-Donahue CA,Feinberg AP','Nature genetics','During the progression of pancreatic ductal adenocarcinoma (PDAC), heterogeneous subclonal populations emerge that drive primary tumor growth, regional spread, distant metastasis, and patient death. However, the genetics of metastases largely reflects that of the primary tumor in untreated patients, and PDAC driver mutations are shared by all subclones. This raises the possibility that an epigenetic process might operate during metastasis. Here we report large-scale reprogramming of chromatin modifications during the natural evolution of distant metastasis. Changes were targeted to thousands of large chromatin domains across the genome that collectively specified malignant traits, including euchromatin and large organized chromatin histone H3 lysine 9 (H3K9)-modified (LOCK) heterochromatin. Remarkably, distant metastases co-evolved a dependence on the oxidative branch of the pentose phosphate pathway (oxPPP), and oxPPP inhibition selectively reversed reprogrammed chromatin, malignant gene expression programs, and tumorigenesis. These findings suggest a model whereby linked metabolic-epigenetic programs are selected for enhanced tumorigenic fitness during the evolution of distant metastasis.\n','2017-01-16',0,''),(337188,'11742071','Multiclass cancer diagnosis using tumor gene expression signatures.','Ramaswamy S,Tamayo P,Rifkin R,Mukherjee S,Yeang CH,Angelo M,Ladd C,Reich M,Latulippe E,Mesirov JP,Poggio T,Gerald W,Loda M,Lander ES,Golub TR','Proceedings of the National Academy of Sciences of the United States of America','The optimal treatment of patients with cancer depends on establishing accurate diagnoses by using a complex combination of clinical and histopathological data. In some instances, this task is difficult or impossible because of atypical clinical presentation or histopathology. To determine whether the diagnosis of multiple common adult malignancies could be achieved purely by molecular classification, we subjected 218 tumor samples, spanning 14 common tumor types, and 90 normal tissue samples to oligonucleotide microarray gene expression analysis. The expression levels of 16,063 genes and expressed sequence tags were used to evaluate the accuracy of a multiclass classifier based on a support vector machine algorithm. Overall classification accuracy was 78%, far exceeding the accuracy of random classification (9%). Poorly differentiated cancers resulted in low-confidence predictions and could not be accurately classified according to their tissue of origin, indicating that they are molecularly distinct entities with dramatically different gene expression patterns compared with their well differentiated counterparts. Taken together, these results demonstrate the feasibility of accurate, multiclass molecular cancer classification and suggest a strategy for future clinical implementation of molecular cancer diagnostics.\n','2001-12-25',0,''),(337189,'17767167','Death-receptor O-glycosylation controls tumor-cell sensitivity to the proapoptotic ligand Apo2L/TRAIL.','Wagner KW,Punnoose EA,Januario T,Lawrence DA,Pitti RM,Lancaster K,Lee D,von Goetz M,Yee SF,Totpal K,Huw L,Katta V,Cavet G,Hymowitz SG,Amler L,Ashkenazi A','Nature medicine','Apo2L/TRAIL stimulates cancer cell death through the proapoptotic receptors DR4 and DR5, but the determinants of tumor susceptibility to this ligand are not fully defined. mRNA expression of the peptidyl O-glycosyltransferase GALNT14 correlated with Apo2L/TRAIL sensitivity in pancreatic carcinoma, non-small-cell lung carcinoma and melanoma cell lines, and up to 30% of samples from various human malignancies showed GALNT14 overexpression. RNA interference of GALNT14 reduced cellular Apo2L/TRAIL sensitivity, whereas overexpression increased responsiveness. Biochemical analysis of DR5 identified several ectodomain O-(N-acetyl galactosamine-galactose-sialic acid) structures. Sequence comparison predicted conserved extracellular DR4 and DR5 O-glycosylation sites; progressive mutation of the DR5 sites attenuated apoptotic signaling. O-glycosylation promoted ligand-stimulated clustering of DR4 and DR5, which mediated recruitment and activation of the apoptosis-initiating protease caspase-8. These results uncover a new link between death-receptor O-glycosylation and apoptotic signaling, providing potential predictive biomarkers for Apo2L/TRAIL-based cancer therapy.\n','2007-09-10',0,''),(337190,'21892151','Toward the blood-borne miRNome of human diseases.','Keller A,Leidinger P,Bauer A,Elsharawy A,Haas J,Backes C,Wendschlag A,Giese N,Tjaden C,Ott K,Werner J,Hackert T,Ruprecht K,Huwer H,Huebers J,Jacobs G,Rosenstiel P,Dommisch H,Schaefer A,Müller-Quernheim J,Wullich B,Keck B,Graf N,Reichrath J,Vogel B,Nebel A,Jager SU,Staehler P,Amarantos I,Boisguerin V,Staehler C,Beier M,Scheffler M,Büchler MW,Wischhusen J,Haeusler SF,Dietl J,Hofmann S,Lenhof HP,Schreiber S,Katus HA,Rottbauer W,Meder B,Hoheisel JD,Franke A,Meese E','Nature methods','In a multicenter study, we determined the expression profiles of 863 microRNAs by array analysis of 454 blood samples from human individuals with different cancers or noncancer diseases, and validated this &apo;miRNome&apo; by quantitative real-time PCR. We detected consistently deregulated profiles for all tested diseases; pathway analysis confirmed disease association of the respective microRNAs. We observed significant correlations (P = 0.004) between the genomic location of disease-associated genetic variants and deregulated microRNAs.\n','2011-09-30',0,''),(337192,'26063736','The landscape of antisense gene expression in human cancers.','Balbin OA,Malik R,Dhanasekaran SM,Prensner JR,Cao X,Wu YM,Robinson D,Wang R,Chen G,Beer DG,Nesvizhskii AI,Chinnaiyan AM','Genome research','High-throughput RNA sequencing has revealed more pervasive transcription of the human genome than previously anticipated. However, the extent of natural antisense transcripts&apo; (NATs) expression, their regulation of cognate sense genes, and the role of NATs in cancer remain poorly understood. Here, we use strand-specific paired-end RNA sequencing (ssRNA-seq) data from 376 cancer samples covering nine tissue types to comprehensively characterize the landscape of antisense expression. We found consistent antisense expression in at least 38% of annotated transcripts, which in general is positively correlated with sense gene expression. Investigation of sense/antisense pair expressions across tissue types revealed lineage-specific, ubiquitous and cancer-specific antisense loci transcription. Comparisons between tumor and normal samples identified both concordant (same direction) and discordant (opposite direction) sense/antisense expression patterns. Finally, we provide OncoNAT, a catalog of cancer-related genes with significant antisense transcription, which will enable future investigations of sense/antisense regulation in cancer. Using OncoNAT we identified several functional NATs, including NKX2-1-AS1 that regulates the NKX2-1 oncogene and cell proliferation in lung cancer cells. Overall, this study provides a comprehensive account of NATs and supports a role for NATs&apo; regulation of tumor suppressors and oncogenes in cancer biology.\n© 2015 Balbin et al.; Published by Cold Spring Harbor Laboratory Press.\n','2015-07-03',0,''),(337193,'26479923','High-throughput screening using patient-derived tumor xenografts to predict clinical trial drug response.','Gao H,Korn JM,Ferretti S,Monahan JE,Wang Y,Singh M,Zhang C,Schnell C,Yang G,Zhang Y,Balbin OA,Barbe S,Cai H,Casey F,Chatterjee S,Chiang DY,Chuai S,Cogan SM,Collins SD,Dammassa E,Ebel N,Embry M,Green J,Kauffmann A,Kowal C,Leary RJ,Lehar J,Liang Y,Loo A,Lorenzana E,Robert McDonald E,McLaughlin ME,Merkin J,Meyer R,Naylor TL,Patawaran M,Reddy A,Röelli C,Ruddy DA,Salangsang F,Santacroce F,Singh AP,Tang Y,Tinetto W,Tobler S,Velazquez R,Venkatesan K,Von Arx F,Wang HQ,Wang Z,Wiesmann M,Wyss D,Xu F,Bitter H,Atadja P,Lees E,Hofmann F,Li E,Keen N,Cozens R,Jensen MR,Pryer NK,Williams JA,Sellers WR','Nature medicine','Profiling candidate therapeutics with limited cancer models during preclinical development hinders predictions of clinical efficacy and identifying factors that underlie heterogeneous patient responses for patient-selection strategies. We established ∼1,000 patient-derived tumor xenograft models (PDXs) with a diverse set of driver mutations. With these PDXs, we performed in vivo compound screens using a 1 × 1 × 1 experimental design (PDX clinical trial or PCT) to assess the population responses to 62 treatments across six indications. We demonstrate both the reproducibility and the clinical translatability of this approach by identifying associations between a genotype and drug response, and established mechanisms of resistance. In addition, our results suggest that PCTs may represent a more accurate approach than cell line models for assessing the clinical potential of some therapeutic modalities. We therefore propose that this experimental paradigm could potentially improve preclinical evaluation of treatment modalities and enhance our ability to predict clinical trial responses.\n','2015-11-06',0,'PCA,Tumour purity,Gene expression,Correlations'),(337194,'25587357','A multi-gene signature predicts outcome in patients with pancreatic ductal adenocarcinoma','Haider S, Wang J, Nagano A, Desai A, Arumugam P, Dumartin L, Fitzgibbon J, Hagemann T, Marshall JF, Kocher HM, Crnogorac-Jurcevic T, Scarpa A, Lemoine NR, Chelala C','Genome Medicine','Improved usage of the repertoires of pancreatic ductal adenocarcinoma (PDAC) profiles is crucially needed to guide the development of predictive and prognostic tools that could inform the selection of treatment options.Using publicly available mRNA abundance datasets, we performed a large retrospective meta-analysis on 466 PDAC patients to discover prognostic gene signatures. These signatures were trained on two clinical cohorts (n = 70), and validated on four independent clinical cohorts (n = 246). Further validation of the identified gene signature was performed using quantitative real-time RT-PCR. We identified 225 candidate prognostic genes. Using these, a 36-gene signature was discovered and validated on fully independent clinical cohorts (hazard ratio (HR) = 2.06, 95% confidence interval (CI) = 1.51 to 2.81, P = 3.62 × 10(-6), n = 246). This signature serves as a good alternative prognostic stratification marker compared to tumour grade (HR = 2.05, 95% CI = 1.45 to 2.88, P = 3.18 × 10(-5)) and tumour node metastasis (TNM) stage (HR = 1.13, 95% CI = 0.66 to 1.94, P = 0.67). Upon multivariate analysis with adjustment for TNM stage and tumour grade, the 36-gene signature remained an independent prognostic predictor of clinical outcome (HR = 2.21, 95% CI = 1.17 to 4.16, P = 0.01). Univariate assessment revealed higher expression of ITGA5, SEMA3A, KIF4A, IL20RB, SLC20A1, CDC45, PXN, SSX3 and TMEM26 was correlated with shorter survival while B3GNT1, NOSTRIN and CADPS down-regulation was associated with poor outcome.','2014-12-03',0,'PCA,Gene expression,Correlations,Survival analysis');
/*!40000 ALTER TABLE `Articles` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Articles_Keywords`
--

DROP TABLE IF EXISTS `Articles_Keywords`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Articles_Keywords` (
  `PMID` varchar(20) COLLATE utf8_bin NOT NULL,
  `KW` int(11) NOT NULL,
  PRIMARY KEY (`PMID`,`KW`),
  KEY `KW` (`KW`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Articles_Keywords`
--

LOCK TABLES `Articles_Keywords` WRITE;
/*!40000 ALTER TABLE `Articles_Keywords` DISABLE KEYS */;
INSERT INTO `Articles_Keywords` VALUES ('22363658',16474),('23372777',16474),('25579086',16474),('26525104',16474),('11742071',16475),('19260470',16475),('19931263',16475),('23372777',16475),('24498386',16475),('25579086',16475),('26525104',16475),('26525104',16476),('16053509',16477),('18754008',16477),('19931263',16477),('21460848',16477),('22261810',16477),('22273699',16477),('22363658',16477),('25579086',16477),('25607660',16477),('26168401',16477),('26343385',16477),('26479923',16477),('26525104',16477),('27281208',16477),('11742071',16478),('16053509',16478),('17699763',16478),('18754008',16478),('18765825',16478),('18852131',16478),('19931263',16478),('20209107',16478),('20644708',16478),('21460848',16478),('21892151',16478),('22261810',16478),('22273699',16478),('23007696',16478),('23372777',16478),('24204737',16478),('24498386',16478),('25259922',16478),('25579086',16478),('25601205',16478),('25607660',16478),('26063736',16478),('26525104',16478),('26769127',16478),('26525104',16479),('11742071',16480),('16053509',16480),('17699763',16480),('17767167',16480),('18413752',16480),('18754008',16480),('18765825',16480),('18852131',16480),('19260470',16480),('19732725',16480),('19931263',16480),('20209107',16480),('20644708',16480),('20885998',16480),('21460848',16480),('21892151',16480),('22261810',16480),('22273699',16480),('22363658',16480),('22763454',16480),('23007696',16480),('23372777',16480),('24204737',16480),('24498386',16480),('25242334',16480),('25259922',16480),('25336113',16480),('25579086',16480),('25601205',16480),('25607660',16480),('26063736',16480),('26168401',16480),('26343385',16480),('26479923',16480),('26525104',16480),('26769127',16480),('27281208',16480),('19931263',16481),('22261810',16481),('22273699',16481),('22363658',16481),('23372777',16481),('25579086',16481),('25607660',16481),('26525104',16481),('24204737',16482),('24498386',16482),('26525104',16482),('11742071',16483),('18754008',16483),('19732725',16483),('26063736',16483),('26479923',16483),('26525104',16483),('26525104',16484),('22261810',16485),('26525104',16485),('26525104',16486),('21460848',16487),('26525104',16487),('18413752',16488),('24204737',16488),('25336113',16488),('26525104',16488),('26525104',16489),('18852131',16490),('22261810',16490),('25259922',16490),('25336113',16490),('25601205',16490),('26525104',16490),('27281208',16490),('26525104',16491),('26525104',16492),('16053509',16493),('20644708',16493),('21460848',16493),('22261810',16493),('22273699',16493),('22363658',16493),('23007696',16493),('24204737',16493),('24498386',16493),('25259922',16493),('25336113',16493),('25607660',16493),('26168401',16493),('26343385',16493),('26479923',16493),('26769127',16493),('27281208',16493),('16053509',16494),('16053509',16495),('18413752',16495),('18754008',16495),('18765825',16495),('19260470',16495),('19931263',16495),('20209107',16495),('20644708',16495),('20885998',16495),('22363658',16495),('22763454',16495),('24204737',16495),('25242334',16495),('25336113',16495),('25579086',16495),('25607660',16495),('26063736',16495),('26168401',16495),('16053509',16496),('17699763',16496),('17767167',16496),('18413752',16496),('18765825',16496),('18852131',16496),('19260470',16496),('19931263',16496),('20644708',16496),('21892151',16496),('22273699',16496),('22363658',16496),('24204737',16496),('24498386',16496),('25607660',16496),('16053509',16497),('16053509',16498),('17699763',16498),('17767167',16498),('18413752',16498),('18765825',16498),('18852131',16498),('19260470',16498),('19732725',16498),('19931263',16498),('20209107',16498),('20644708',16498),('20885998',16498),('21460848',16498),('22261810',16498),('22273699',16498),('22363658',16498),('22763454',16498),('23007696',16498),('23372777',16498),('24204737',16498),('24498386',16498),('25242334',16498),('25259922',16498),('25336113',16498),('25579086',16498),('25607660',16498),('26168401',16498),('26343385',16498),('26479923',16498),('27281208',16498),('17699763',16499),('17767167',16499),('18413752',16499),('18754008',16499),('20209107',16499),('21460848',16499),('22763454',16499),('23007696',16499),('23402223',16499),('24204737',16499),('25242334',16499),('25259922',16499),('25336113',16499),('26168401',16499),('26343385',16499),('26479923',16499),('27281208',16499),('17699763',16500),('17767167',16500),('18765825',16500),('19732725',16500),('20209107',16500),('20885998',16500),('21460848',16500),('22363658',16500),('24204737',16500),('25259922',16500),('26168401',16500),('26769127',16500),('17699763',16501),('18413752',16502),('18413752',16503),('18413752',16504),('18852131',16504),('19732725',16504),('21460848',16504),('22363658',16504),('25259922',16504),('26479923',16504),('18413752',16505),('18413752',16506),('21460848',16506),('18413752',16507),('18754008',16507),('24204737',16507),('26168401',16507),('26479923',16507),('18413752',16508),('21460848',16508),('17767167',16509),('18413752',16509),('24204737',16509),('18754008',16510),('19732725',16510),('25601205',16510),('18754008',16511),('18754008',16512),('18765825',16512),('18754008',16513),('18754008',16514),('18754008',16515),('18754008',16516),('25259922',16516),('26343385',16516),('18765825',16517),('18765825',16518),('18765825',16519),('22763454',16519),('18765825',16520),('18852131',16520),('23007696',16520),('18765825',16521),('20644708',16521),('17767167',16522),('18852131',16522),('18852131',16523),('18852131',16524),('23007696',16524),('24498386',16524),('18852131',16525),('20209107',16525),('24498386',16525),('25601205',16525),('18852131',16526),('18852131',16527),('18852131',16528),('18852131',16529),('18852131',16530),('18852131',16531),('17767167',16532),('18852131',16532),('25259922',16532),('18852131',16533),('19260470',16533),('23007696',16533),('18852131',16534),('18852131',16535),('26479923',16535),('27281208',16535),('19260470',16536),('19260470',16537),('20644708',16537),('24204737',16537),('25259922',16537),('25336113',16537),('25579086',16537),('26343385',16537),('26769127',16537),('19260470',16538),('19260470',16539),('19260470',16540),('19260470',16541),('19260470',16542),('19260470',16543),('19260470',16544),('19732725',16545),('19732725',16546),('19732725',16547),('19732725',16548),('19732725',16549),('19732725',16550),('22363658',16550),('19732725',16551),('19732725',16552),('19732725',16553),('19732725',16554),('19931263',16555),('19931263',16556),('19931263',16557),('17767167',16558),('19931263',16558),('22261810',16558),('22763454',16558),('24498386',16558),('25579086',16558),('19931263',16559),('19931263',16560),('19931263',16561),('20209107',16562),('20209107',16563),('21460848',16563),('22363658',16563),('20209107',16564),('22763454',16564),('25259922',16564),('26479923',16564),('27281208',16564),('20209107',16565),('20644708',16566),('22763454',16566),('23007696',16566),('20644708',16567),('23372777',16567),('20644708',16568),('22261810',16568),('25579086',16568),('20644708',16569),('22261810',16569),('20885998',16570),('17767167',16571),('20885998',16571),('20885998',16572),('20885998',16573),('20885998',16574),('21460848',16575),('21460848',16576),('21460848',16577),('11742071',16578),('22261810',16578),('23372777',16578),('24204737',16578),('25579086',16578),('26063736',16578),('22261810',16579),('22273699',16580),('24498386',16580),('25607660',16580),('22273699',16581),('25607660',16581),('22273699',16582),('23372777',16582),('27281208',16582),('22273699',16583),('26769127',16583),('22273699',16584),('22273699',16585),('22363658',16585),('22273699',16586),('22273699',16587),('24498386',16587),('22363658',16588),('22363658',16589),('22363658',16590),('22363658',16591),('22363658',16592),('22363658',16593),('22363658',16594),('25579086',16594),('22363658',16595),('17767167',16596),('22763454',16596),('22763454',16597),('22763454',16598),('25242334',16598),('27281208',16598),('22763454',16599),('23007696',16600),('21892151',16601),('23007696',16601),('24498386',16601),('23007696',16602),('23372777',16603),('26479923',16603),('23372777',16604),('24204737',16605),('24204737',16606),('25579086',16606),('24204737',16607),('24204737',16608),('24204737',16609),('17767167',16610),('24498386',16610),('24498386',16611),('24498386',16612),('24498386',16613),('24498386',16614),('24498386',16615),('25242334',16615),('25242334',16616),('25242334',16617),('25242334',16618),('25242334',16619),('25242334',16620),('25242334',16621),('25259922',16622),('25259922',16623),('25259922',16624),('25336113',16625),('27281208',16625),('25336113',16626),('25579086',16627),('25601205',16628),('25601205',16629),('25601205',16630),('25601205',16631),('25601205',16632),('25601205',16633),('25601205',16634),('26343385',16634),('25601205',16635),('25601205',16636),('25601205',16637),('25601205',16638),('25601205',16639),('25601205',16640),('25601205',16641),('25601205',16642),('26063736',16642),('25607660',16643),('25607660',16644),('25607660',16645),('25607660',16646),('26168401',16647),('26168401',16648),('26168401',16649),('26168401',16650),('26168401',16651),('26168401',16652),('26343385',16652),('26168401',16653),('26168401',16654),('26168401',16655),('26769127',16655),('26168401',16656),('26343385',16657),('26769127',16658),('26769127',16659),('27281208',16660),('27281208',16661),('27281208',16662),('23402223',16663),('27281208',16663),('27281208',16664),('27281208',16665),('27281208',16666),('27281208',16667),('27281208',16668),('11742071',16669),('17767167',16670),('26479923',16670),('17767167',16671),('26479923',16671),('17767167',16672),('26479923',16672),('17767167',16673),('17767167',16674),('17767167',16675),('21892151',16676),('21892151',16677),('23402223',16678),('23402223',16679),('23402223',16680),('23402223',16681),('23402223',16682),('26063736',16683),('26063736',16684),('26063736',16685),('26063736',16686),('26479923',16687),('26479923',16688),('26479923',16689),('26479923',16690),('26479923',16691);
/*!40000 ALTER TABLE `Articles_Keywords` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Downloads`
--

DROP TABLE IF EXISTS `Downloads`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Downloads` (
  `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `full_name` varchar(50) COLLATE utf8_bin NOT NULL,
  `email` varchar(50) COLLATE utf8_bin NOT NULL,
  `institution` varchar(50) COLLATE utf8_bin NOT NULL,
  `research` text COLLATE utf8_bin NOT NULL,
  PRIMARY KEY (`ID`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Downloads`
--

LOCK TABLES `Downloads` WRITE;
/*!40000 ALTER TABLE `Downloads` DISABLE KEYS */;
/*!40000 ALTER TABLE `Downloads` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Keywords`
--

DROP TABLE IF EXISTS `Keywords`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Keywords` (
  `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `Keyword` text COLLATE utf8_bin NOT NULL,
  `Occurrences` int(11) NOT NULL,
  `Global_Occurrences` bigint(20) NOT NULL,
  `Level` int(11) NOT NULL,
  `Rank` int(11) NOT NULL,
  PRIMARY KEY (`ID`)
) ENGINE=InnoDB AUTO_INCREMENT=16692 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Keywords`
--

LOCK TABLES `Keywords` WRITE;
/*!40000 ALTER TABLE `Keywords` DISABLE KEYS */;
INSERT INTO `Keywords` VALUES (16474,'adult',0,0,0,0),(16475,'biomarkers, tumor',0,0,0,0),(16476,'blood platelets',0,0,0,0),(16477,'female',0,0,0,0),(16478,'gene expression profiling',0,0,0,0),(16479,'gene ontology',0,0,0,0),(16480,'humans',0,0,0,0),(16481,'middle aged',0,0,0,0),(16482,'mutation',0,0,0,0),(16483,'neoplasms',0,0,0,0),(16484,'pathology, molecular',0,0,0,0),(16485,'phosphatidylinositol 3-kinases',0,0,0,0),(16486,'proto-oncogene proteins c-met',0,0,0,0),(16487,'proto-oncogene proteins p21(ras)',0,0,0,0),(16488,'receptor, epidermal growth factor',0,0,0,0),(16489,'receptor, erbb-2',0,0,0,0),(16490,'signal transduction',0,0,0,0),(16491,'support vector machine',0,0,0,0),(16492,'young adult',0,0,0,0),(16493,'carcinoma, pancreatic ductal',0,0,0,0),(16494,'case-control studies',0,0,0,0),(16495,'gene expression regulation, neoplastic',0,0,0,0),(16496,'oligonucleotide array sequence analysis',0,0,0,0),(16497,'pancreatic juice',0,0,0,0),(16498,'pancreatic neoplasms',0,0,0,0),(16499,'animals',0,0,0,0),(16500,'cell line, tumor',0,0,0,0),(16501,'up-regulation',0,0,0,0),(16502,'antibodies, monoclonal',0,0,0,0),(16503,'antibodies, monoclonal, humanized',0,0,0,0),(16504,'antineoplastic agents',0,0,0,0),(16505,'cetuximab',0,0,0,0),(16506,'erlotinib hydrochloride',0,0,0,0),(16507,'neoplasm transplantation',0,0,0,0),(16508,'quinazolines',0,0,0,0),(16509,'transplantation, heterologous',0,0,0,0),(16510,'cell line',0,0,0,0),(16511,'hedgehog proteins',0,0,0,0),(16512,'ligands',0,0,0,0),(16513,'paracrine communication',0,0,0,0),(16514,'receptors, g-protein-coupled',0,0,0,0),(16515,'smoothened receptor',0,0,0,0),(16516,'stromal cells',0,0,0,0),(16517,'algorithms',0,0,0,0),(16518,'cell membrane',0,0,0,0),(16519,'genes, neoplasm',0,0,0,0),(16520,'immunohistochemistry',0,0,0,0),(16521,'tissue array analysis',0,0,0,0),(16522,'amino acid sequence',0,0,0,0),(16523,'benzoquinones',0,0,0,0),(16524,'blotting, western',0,0,0,0),(16525,'cell proliferation',0,0,0,0),(16526,'dose-response relationship, drug',0,0,0,0),(16527,'down-regulation',0,0,0,0),(16528,'hsp90 heat-shock proteins',0,0,0,0),(16529,'isotope labeling',0,0,0,0),(16530,'lactams, macrocyclic',0,0,0,0),(16531,'mass spectrometry',0,0,0,0),(16532,'molecular sequence data',0,0,0,0),(16533,'neoplasm proteins',0,0,0,0),(16534,'time factors',0,0,0,0),(16535,'xenograft model antitumor assays',0,0,0,0),(16536,'14-3-3 proteins',0,0,0,0),(16537,'adenocarcinoma',0,0,0,0),(16538,'exonucleases',0,0,0,0),(16539,'exoribonucleases',0,0,0,0),(16540,'keratin-7',0,0,0,0),(16541,'laminin',0,0,0,0),(16542,'membrane proteins',0,0,0,0),(16543,'phosphofructokinases',0,0,0,0),(16544,'protein-serine-threonine kinases',0,0,0,0),(16545,'antimetabolites, antineoplastic',0,0,0,0),(16546,'antineoplastic agents, phytogenic',0,0,0,0),(16547,'cytarabine',0,0,0,0),(16548,'drug therapy',0,0,0,0),(16549,'kidney',0,0,0,0),(16550,'nuclear proteins',0,0,0,0),(16551,'paclitaxel',0,0,0,0),(16552,'phosphoprotein phosphatases',0,0,0,0),(16553,'proto-oncogene proteins c-akt',0,0,0,0),(16554,'taxoids',0,0,0,0),(16555,'genetic testing',0,0,0,0),(16556,'logistic models',0,0,0,0),(16557,'prospective studies',0,0,0,0),(16558,'rna, messenger',0,0,0,0),(16559,'retrospective studies',0,0,0,0),(16560,'risk assessment',0,0,0,0),(16561,'saliva',0,0,0,0),(16562,'antineoplastic combined chemotherapy protocols',0,0,0,0),(16563,'deoxycytidine',0,0,0,0),(16564,'disease models, animal',0,0,0,0),(16565,'thiazoles',0,0,0,0),(16566,'neoplasm metastasis',0,0,0,0),(16567,'neoplasm staging',0,0,0,0),(16568,'prognosis',0,0,0,0),(16569,'survival analysis',0,0,0,0),(16570,'epithelial-mesenchymal transition',0,0,0,0),(16571,'glycosylation',0,0,0,0),(16572,'models, biological',0,0,0,0),(16573,'polysaccharides',0,0,0,0),(16574,'proteins',0,0,0,0),(16575,'gata6 transcription factor',0,0,0,0),(16576,'pharmacogenetics',0,0,0,0),(16577,'proto-oncogene proteins',0,0,0,0),(16578,'cluster analysis',0,0,0,0),(16579,'src-family kinases',0,0,0,0),(16580,'adenocarcinoma, mucinous',0,0,0,0),(16581,'adenocarcinoma, papillary',0,0,0,0),(16582,'disease progression',0,0,0,0),(16583,'genetic markers',0,0,0,0),(16584,'neoplasm grading',0,0,0,0),(16585,'neoplasm invasiveness',0,0,0,0),(16586,'precancerous conditions',0,0,0,0),(16587,'real-time polymerase chain reaction',0,0,0,0),(16588,'cell cycle proteins',0,0,0,0),(16589,'cohort studies',0,0,0,0),(16590,'dipeptidases',0,0,0,0),(16591,'gpi-linked proteins',0,0,0,0),(16592,'genetic association studies',0,0,0,0),(16593,'maryland',0,0,0,0),(16594,'proportional hazards models',0,0,0,0),(16595,'treatment outcome',0,0,0,0),(16596,'cell survival',0,0,0,0),(16597,'contact inhibition',0,0,0,0),(16598,'neoplastic cells, circulating',0,0,0,0),(16599,'wnt2 protein',0,0,0,0),(16600,'calcium-binding proteins',0,0,0,0),(16601,'reverse transcriptase polymerase chain reaction',0,0,0,0),(16602,'zebrafish',0,0,0,0),(16603,'carcinoma',0,0,0,0),(16604,'pedigree',0,0,0,0),(16605,'cytokines',0,0,0,0),(16606,'kaplan-meier estimate',0,0,0,0),(16607,'liver neoplasms',0,0,0,0),(16608,'peritoneal neoplasms',0,0,0,0),(16609,'smad4 protein',0,0,0,0),(16610,'apoptosis',0,0,0,0),(16611,'carcinoma, papillary',0,0,0,0),(16612,'cell cycle',0,0,0,0),(16613,'chromogranins',0,0,0,0),(16614,'mucins',0,0,0,0),(16615,'tumor cells, cultured',0,0,0,0),(16616,'carrier proteins',0,0,0,0),(16617,'cell movement',0,0,0,0),(16618,'extracellular matrix',0,0,0,0),(16619,'osteonectin',0,0,0,0),(16620,'rna, small interfering',0,0,0,0),(16621,'retinal dehydrogenase',0,0,0,0),(16622,'calcitriol',0,0,0,0),(16623,'pancreatitis',0,0,0,0),(16624,'receptors, calcitriol',0,0,0,0),(16625,'cell transformation, neoplastic',0,0,0,0),(16626,'sox9 transcription factor',0,0,0,0),(16627,'intestinal neoplasms',0,0,0,0),(16628,'cell communication',0,0,0,0),(16629,'cell differentiation',0,0,0,0),(16630,'coculture techniques',0,0,0,0),(16631,'embryonic stem cells',0,0,0,0),(16632,'endothelial cells',0,0,0,0),(16633,'endothelial growth factors',0,0,0,0),(16634,'gene expression',0,0,0,0),(16635,'gene expression regulation, developmental',0,0,0,0),(16636,'homeodomain proteins',0,0,0,0),(16637,'immunophenotyping',0,0,0,0),(16638,'pancreas',0,0,0,0),(16639,'phenotype',0,0,0,0),(16640,'stem cell niche',0,0,0,0),(16641,'trans-activators',0,0,0,0),(16642,'transcriptome',0,0,0,0),(16643,'diagnosis, differential',0,0,0,0),(16644,'gene regulatory networks',0,0,0,0),(16645,'pilot projects',0,0,0,0),(16646,'serum albumin',0,0,0,0),(16647,'active transport, cell nucleus',0,0,0,0),(16648,'amino acids',0,0,0,0),(16649,'autophagy',0,0,0,0),(16650,'basic helix-loop-helix leucine zipper transcription factors',0,0,0,0),(16651,'energy metabolism',0,0,0,0),(16652,'heterografts',0,0,0,0),(16653,'homeostasis',0,0,0,0),(16654,'lysosomes',0,0,0,0),(16655,'transcription factors',0,0,0,0),(16656,'transcription, genetic',0,0,0,0),(16657,'genetic heterogeneity',0,0,0,0),(16658,'epigenesis, genetic',0,0,0,0),(16659,'gene expression regulation',0,0,0,0),(16660,'drug resistance, neoplasm',0,0,0,0),(16661,'gene deletion',0,0,0,0),(16662,'genes, reporter',0,0,0,0),(16663,'models, genetic',0,0,0,0),(16664,'molecular imaging',0,0,0,0),(16665,'nerve tissue proteins',0,0,0,0),(16666,'oligonucleotides, antisense',0,0,0,0),(16667,'rna-binding proteins',0,0,0,0),(16668,'survival rate',0,0,0,0),(16669,'multigene family',0,0,0,0),(16670,'carcinoma, non-small-cell lung',0,0,0,0),(16671,'lung neoplasms',0,0,0,0),(16672,'melanoma',0,0,0,0),(16673,'receptors, death domain',0,0,0,0),(16674,'receptors, tnf-related apoptosis-inducing ligand',0,0,0,0),(16675,'tnf-related apoptosis-inducing ligand',0,0,0,0),(16676,'disease',0,0,0,0),(16677,'genetic variation',0,0,0,0),(16678,'adaptation, physiological',0,0,0,0),(16679,'evolution, molecular',0,0,0,0),(16680,'genetics, population',0,0,0,0),(16681,'passeriformes',0,0,0,0),(16682,'sequence homology, nucleic acid',0,0,0,0),(16683,'computational biology',0,0,0,0),(16684,'genetic loci',0,0,0,0),(16685,'organ specificity',0,0,0,0),(16686,'rna, antisense',0,0,0,0),(16687,'breast neoplasms',0,0,0,0),(16688,'colorectal neoplasms',0,0,0,0),(16689,'high-throughput screening assays',0,0,0,0),(16690,'skin neoplasms',0,0,0,0),(16691,'stomach neoplasms',0,0,0,0);
/*!40000 ALTER TABLE `Keywords` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `ccle`
--

DROP TABLE IF EXISTS `ccle`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `ccle` (
  `name` varchar(50) DEFAULT NULL,
  `target` varchar(100) DEFAULT NULL,
  `target_details` varchar(100) DEFAULT NULL,
  `cell_type` varchar(100) DEFAULT NULL,
  `type_derived_from` varchar(100) DEFAULT NULL,
  `gender` varchar(20) DEFAULT NULL,
  `ethnicity` varchar(50) DEFAULT NULL,
  `age` varchar(10) DEFAULT NULL,
  `er` varchar(10) DEFAULT NULL,
  `pr` varchar(10) DEFAULT NULL,
  `her2` varchar(10) DEFAULT NULL,
  `notes` text
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `ccle`
--

LOCK TABLES `ccle` WRITE;
/*!40000 ALTER TABLE `ccle` DISABLE KEYS */;
INSERT INTO `ccle` VALUES ('MiaPaCa-2','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','Caucasian','65','NA','NA','NA',NULL),('Capan-2','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','Caucasian','56','NA','NA','NA',NULL),('Hs766T','Adenocarcinoma',NULL,'Metastasis (lymph node)',NULL,'Male','Caucasian','46','NA','NA','NA',NULL),('Panc10.05','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','Caucasian','NA','NA','NA','NA',NULL),('AsPC-1','Ductal adenocarcinoma',NULL,'Metastasis (ascites)',NULL,'Female','Caucasian','62','NA','NA','NA',NULL),('L3.3','Adenosquamous carcinoma',NULL,'Metastasis (lymph node)',NULL,'Female','NS','NA','NA','NA','NA',NULL),('KP-2','Carcinoma',NULL,'Primary Tumour',NULL,'Female','NS','NA','NA','NA','NA',NULL),('PaTu8902','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','NS','NA','NA','NA','NA',NULL),('DAN-G','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','NS','NA','NA','NA','NA',NULL),('HuP-T3','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('TCC-PAN2','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','NS','NA','NA','NA','NA',NULL),('HuP-T4','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('PSN1','Adenocarcinoma',NULL,'Primary Tumour',NULL,'NA','NS','NA','NA','NA','NA',NULL),('PaTu8988T','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('Panc4.03','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','White','70','NA','NA','NA',NULL),('KCI-MOH1','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('PK-59','Carcinoma',NULL,'Metastasis (liver)',NULL,'Female','NS','NA','NA','NA','NA',NULL),('QGP-1','Somatostatinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('Panc3.27','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','White','65','NA','NA','NA',NULL),('HPAF-II','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','Caucasian','44','NA','NA','NA',NULL),('Capan-1','Ductal adenocarcinoma',NULL,'Metastasis (liver)',NULL,'Male','Caucasian','40','NA','NA','NA',NULL),('CFPAC-1','Ductal adenocarcinoma',NULL,'Metastasis (liver)',NULL,'Male','Caucasian','26','NA','NA','NA',NULL),('Panc2.13','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','White','64','NA','NA','NA',NULL),('BxPC-3','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','NS','61','NA','NA','NA',NULL),('KP-3','Adenosquamous carcinoma',NULL,'Metastasis (liver)',NULL,'Male','NS','NA','NA','NA','NA',NULL),('KP-4','Carcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('PaTu8988S','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','NS','NA','NA','NA','NA',NULL),('PL45','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','White','NA','NA','NA','NA',NULL),('SW1990','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','Caucasian','56','NA','NA','NA',NULL),('Panc-1','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','Caucasian','56','NA','NA','NA',NULL),('Panc8.13','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','White','85','NA','NA','NA',NULL),('PK-1','Ductal adenocarcinoma',NULL,'Metastasis (liver)',NULL,'Male','NS','NA','NA','NA','NA',NULL),('Panc5.04','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','White','77','NA','NA','NA',NULL),('SUIT-2','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('YAPC','Carcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('SNU-410','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('T3M-4','Ductal adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('PK-45H','Carcinoma',NULL,'Primary Tumour',NULL,'NA','NS','NA','NA','NA','NA',NULL),('HPAC','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','Caucasian','64','NA','NA','NA',NULL),('SU-86-86','Adenocarcinoma',NULL,'Metastasis (liver)',NULL,'Female','Caucasian','57','NA','NA','NA',NULL),('Panc2.03','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Female','White','70','NA','NA','NA',NULL),('SNU-324','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('SNU-213','Adenocarcinoma',NULL,'Primary Tumour',NULL,'Male','NS','NA','NA','NA','NA',NULL),('KLM-1','Ductal adenocarcinoma',NULL,'Metastasis (liver)',NULL,'Male','NS','NA','NA','NA','NA',NULL);
/*!40000 ALTER TABLE `ccle` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `tcga`
--

DROP TABLE IF EXISTS `tcga`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `tcga` (
  `name` varchar(50) DEFAULT NULL,
  `target` varchar(50) DEFAULT NULL,
  `age` varchar(10) DEFAULT NULL,
  `years_smoked` varchar(10) DEFAULT NULL,
  `alcohol_history` varchar(10) DEFAULT NULL,
  `gender` varchar(30) DEFAULT NULL,
  `ethnicity` varchar(50) DEFAULT NULL,
  `tumor_stage` varchar(10) DEFAULT NULL,
  `tnm_staging` varchar(10) DEFAULT NULL,
  `histologic_grade` varchar(10) DEFAULT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `tcga`
--

LOCK TABLES `tcga` WRITE;
/*!40000 ALTER TABLE `tcga` DISABLE KEYS */;
INSERT INTO `tcga` VALUES ('TCGA-H6-8124-01A','PDAC','57','7','yes','Female','white','IIB','T3N1M0','G3'),('TCGA-IB-AAUT-01A','PDAC','65','15','yes','Male','white','IIB','T2N1M0','G1'),('TCGA-F2-6879-01A','PDAC','57','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-IB-AAUU-01A','PDAC','36','NA','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-HV-A5A5-01A','PDAC','61','NA','yes','Female','asian','IIB','T3N1M0','G2'),('TCGA-2L-AAQI-01A','PDAC','67','NA','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-HV-A7OL-01A','PDAC','71','NA','yes','Male','asian','IIA','T3N0MX','G2'),('TCGA-3A-A9IO-01A','PDAC','56','NA','yes','Male','white','NA','TXNXMX','G1'),('TCGA-Q3-AA2A-01A','PDAC','65','NA','yes','Female','white','IB','T3N1MX','G1'),('TCGA-HZ-7925-01A','PDAC','66','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-H6-8124-11A','Normal','57','7','yes','Female','white','IIB','T3N1M0','G3'),('TCGA-US-A77E-01A','PDAC','73','NA','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-2J-AABE-01A','PDAC','74','10','yes','Male','white','IIA','T3N0M0','G2'),('TCGA-FB-AAQ2-01A','PDAC','81','NA','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-XD-AAUI-01A','PDAC','51','NA','yes','Female','white','IIB','T3N1MX','G2'),('TCGA-HZ-A77P-01A','PDAC','78','NA','no','Male','white','IIB','T3N1MX','G1'),('TCGA-2J-AABR-01A','PDAC','61','NA','no','Female','white','IIA','T3N0M0','G3'),('TCGA-3A-A9IL-01A','PDAC','40','NA','no','Female','white','I','T1MX','G1'),('TCGA-3A-A9I9-01A','PDAC','68','NA','yes','Male','white','IIA','T3N0MX','G2'),('TCGA-HV-A5A6-01A','PDAC','66','NA','NA','Female','asian','IIB','T3N1bM0','G1'),('TCGA-3E-AAAZ-01A','PDAC','72','NA','NA','Male','white','IIA','T3N0MX','G2'),('TCGA-HZ-7923-01A','PDAC','65','NA','NA','Male','white','IIA','T3N0MX','G2'),('TCGA-US-A776-01A','PDAC','61','NA','yes','Male','white','IIA','T3N0MX','G2'),('TCGA-IB-7893-01A','PDAC','65','NA','yes','Male','white','IIA','T3N0M0','G3'),('TCGA-F2-A8YN-01A','PDAC','77','NA','yes','Male','white','IIA','T3N0M0','G2'),('TCGA-FB-AAQ6-01A','PDAC','86','NA','yes','Male','white','IA','T1N0MX','G2'),('TCGA-HZ-A49I-01A','PDAC','78','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-2J-AABP-01A','PDAC','59','NA','yes','Female','white','IIB','T3N1M0','G4'),('TCGA-HZ-8636-01A','PDAC','59','NA','no','Female','white','IV','T3N0M1','G3'),('TCGA-HZ-8001-01A','PDAC','69','NA','NA','Male','white','III','T4N0MX','G2'),('TCGA-IB-7889-01A','PDAC','86','NA','no','Female','white','IIB','T3N1M0','G1'),('TCGA-Z5-AAPL-01A','PDAC','74','NA','NA','Female','white','IIA','T3N0M0','G1'),('TCGA-FB-A545-01A','PDAC','73','NA','no','Female','white','IIB','T3N1M0','G2'),('TCGA-HZ-A49H-01A','PDAC','68','NA','no','Female','white','IIB','T3N1MX','G2'),('TCGA-HZ-8003-01A','PDAC','65','NA','NA','Female','white','IIB','T3N1MX','G2'),('TCGA-XD-AAUL-01A','PDAC','56','NA','yes','Male','white','IIA','T3N0MX','G2'),('TCGA-LB-A8F3-01A','PDAC','65','NA','yes','Female','black or african american','IIA','T3N0MX','G1'),('TCGA-2J-AABO-01A','PDAC','43','10','no','Male','white','IIB','T3N1M0','G2'),('TCGA-IB-A6UG-01A','PDAC','65','NA','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-FB-A7DR-01A','PDAC','48','NA','yes','Male','asian','IIA','T3NXM0','G2'),('TCGA-LB-A7SX-01A','PDAC','75','NA','yes','Female','black or african american','IIB','T3N1MX','G2'),('TCGA-2J-AABF-01A','PDAC','74','5','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-RB-A7B8-01A','PDAC','81','NA','yes','Female','white','IIB','T3N1MX','G2'),('TCGA-FB-AAPZ-01A','PDAC','54','NA','no','Male','white','IIB','T2N1MX','G3'),('TCGA-IB-A5ST-01A','PDAC','65','NA','no','Female','white','IIB','T3N1M0','G2'),('TCGA-IB-7885-01A','PDAC','78','20','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-FB-AAQ1-01A','PDAC','50','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-IB-A5SP-01A','PDAC','78','NA','yes','Male','white','IIA','T3N0M0','G2'),('TCGA-IB-7652-01A','PDAC','50','NA','yes','Female','asian','IIB','T3N1M0','G2'),('TCGA-HZ-8317-01A','PDAC','69','NA','no','Female','white','IIB','T3N1MX','G1'),('TCGA-YH-A8SY-01A','PDAC','74','NA','yes','Female','white','NA','T3N1MX','G2'),('TCGA-IB-7897-01A','PDAC','54','NA','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-2J-AABA-01A','PDAC','55','NA','no','Male','white','IIB','T3N1M0','G2'),('TCGA-IB-AAUV-01A','PDAC','49','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-IB-7647-01A','PDAC','42','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-2L-AAQL-01A','PDAC','82','NA','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-IB-7644-01A','PDAC','66','NA','yes','Female','white','IV','T3N1M1','G2'),('TCGA-IB-AAUO-01A','PDAC','65','NA','yes','Female','white','IIB','T3N1M0','G3'),('TCGA-IB-7888-01A','PDAC','66','NA','no','Female','black or african american','IIA','T3N0M0','G2'),('TCGA-FB-A78T-01A','PDAC','72','20','no','Female','white','IIB','T3N1M0','G2'),('TCGA-2J-AABK-01A','PDAC','72','NA','no','Male','white','IIB','T3N1M0','G2'),('TCGA-M8-A5N4-01A','PDAC','48','NA','yes','Female','not reported','IIA','T3N0M0','G2'),('TCGA-HZ-7918-01A','PDAC','73','NA','no','Male','white','IIB','T3N1M0','G3'),('TCGA-F2-A7TX-01A','PDAC','77','NA','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-HZ-8315-01A','PDAC','54','NA','NA','Female','white','IIA','T3N0MX','G2'),('TCGA-3A-A9IJ-01A','PDAC','65','30','no','Male','white','IB','T2N0MX','G1'),('TCGA-IB-A7LX-01A','PDAC','57','NA','yes','Male','white','IIB','T3N1MX','G2'),('TCGA-PZ-A5RE-01A','PDAC','45','NA','NA','Female','white','IIB','T3N1MX','G3'),('TCGA-L1-A7W4-01A','PDAC','49','NA','yes','Female','white','IIB','T3N1M0','G3'),('TCGA-S4-A8RO-01A','PDAC','76','NA','yes','Female','white','IIB','T3N1MX','G2'),('TCGA-S4-A8RP-01A','PDAC','78','NA','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-US-A77J-01A','PDAC','82','NA','yes','Female','white','IIB','T3N1bMX','G2'),('TCGA-HZ-A77O-01A','PDAC','78','NA','no','Female','white','IIB','T2N1MX','G2'),('TCGA-IB-AAUR-01A','PDAC','67','NA','yes','Male','white','IIB','T3N1M0','G1'),('TCGA-IB-AAUP-01A','PDAC','68','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-US-A774-01A','PDAC','77','37','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-3A-A9J0-01A','PDAC','75','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-US-A779-01A','PDAC','55','NA','yes','Female','white','IIB','T3N1bMX','G1'),('TCGA-HZ-A4BK-01A','PDAC','73','NA','no','Male','white','IIB','T3N1MX','G3'),('TCGA-2L-AAQJ-01A','PDAC','50','NA','yes','Female','white','III','T4N0MX','G2'),('TCGA-HZ-A9TJ-01A','PDAC','71','NA','no','Male','white','IV','T3N0M1','G2'),('TCGA-FB-AAPP-01A','PDAC','72','NA','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-2J-AABV-01A','PDAC','74','NA','no','Male','white','IIB','T3N1M0','G4'),('TCGA-2J-AABT-01A','PDAC','72','NA','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-IB-8127-01A','PDAC','59','40','NA','Male','white','IIB','T3N1M0','G2'),('TCGA-FB-AAPQ-01A','PDAC','66','25','no','Male','white','IIB','T3N1MX','G2'),('TCGA-IB-AAUM-01A','PDAC','77','NA','yes','Male','white','IIB','T2N1M0','G3'),('TCGA-HZ-A8P0-01A','PDAC','76','20','yes','Male','white','IIB','T3N1MX','G1'),('TCGA-XD-AAUG-01A','PDAC','67','NA','yes','Female','white','IV','T3N1M1','G2'),('TCGA-HZ-A49G-01A','PDAC','59','NA','no','Female','white','IIB','T2N1MX','G2'),('TCGA-HV-A5A3-11A','Normal','51','NA','yes','Male','asian','IIA','T3NXMX','G2'),('TCGA-F2-A44G-01A','PDAC','80','NA','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-IB-A7M4-01A','PDAC','82','52','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-HZ-8519-01A','PDAC','74','NA','no','Male','white','IA','T1N0MX','G3'),('TCGA-HZ-A8P1-01A','PDAC','81','15','no','Male','white','IB','T2N0MX','G1'),('TCGA-HZ-8638-01A','PDAC','84','NA','no','Female','white','IA','T1N0MX','GX'),('TCGA-IB-7649-01A','PDAC','73','NA','no','Female','not reported','IIB','T3N1M0','G2'),('TCGA-F2-6880-01A','PDAC','71','NA','yes','Male','white','IIB','T3N1M0','G1'),('TCGA-H8-A6C1-01A','PDAC','54','30','yes','Male','white','IIA','T3N0MX','G2'),('TCGA-F2-7273-01A','PDAC','55','NA','no','Male','black or african american','IIB','T3N1M0','G3'),('TCGA-IB-7887-01A','PDAC','63','NA','no','Female','white','IIB','T3N1M0','G2'),('TCGA-HZ-A77Q-01A','PDAC','56','NA','no','Female','white','IIB','T3N1MX','G2'),('TCGA-H6-A45N-01A','PDAC','89','NA','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-2J-AAB8-01A','PDAC','71','NA','no','Male','white','IIB','T3N1M0','G3'),('TCGA-FB-AAQ3-01A','PDAC','66','NA','yes','Female','white','IIB','T3N1MX','G2'),('TCGA-YY-A8LH-01A','PDAC','61','NA','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-HZ-7924-01A','PDAC','61','NA','no','Female','white','IIA','T3N0MX','G2'),('TCGA-XN-A8T5-01A','PDAC','54','NA','no','Female','white','IB','T2N0M0','G2'),('TCGA-YB-A89D-11A','Normal','60','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-2J-AABH-01A','PDAC','61','NA','no','Male','white','IIA','T3N0M0','G3'),('TCGA-FB-AAPY-01A','PDAC','71','25','yes','Male','white','IIB','T2N1MX','G2'),('TCGA-HZ-A4BH-01A','PDAC','76','NA','no','Male','white','IIB','T3N1bMX','G3'),('TCGA-2L-AAQM-01A','PDAC','53','NA','yes','Male','white','IIB','T3N1MX','G1'),('TCGA-3A-A9I5-01A','PDAC','58','NA','no','Male','white','NA','N0MX','G1'),('TCGA-F2-A44H-01A','PDAC','66','NA','no','Male','white','IIA','T3N0M0','G2'),('TCGA-IB-8126-01A','PDAC','80','4','yes','Female','white','III','T4N1M0','G1'),('TCGA-HV-A5A3-01A','PDAC','51','NA','yes','Male','asian','IIA','T3NXMX','G2'),('TCGA-3A-A9I7-01A','PDAC','46','NA','yes','Male','black or african american','IIB','T3N1MX','G2'),('TCGA-3A-A9IZ-01A','PDAC','47','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-OE-A75W-01A','PDAC','76','20','yes','Male','white','IIA','T3N0M0','G1'),('TCGA-IB-AAUW-01A','PDAC','64','NA','yes','Female','white','IIB','T3N1M0','G3'),('TCGA-FB-AAQ0-01A','PDAC','68','NA','no','Male','white','IIA','T3N0MX','G3'),('TCGA-H6-A45N-11A','Normal','89','NA','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-2J-AAB4-01A','PDAC','49','NA','no','Male','white','IIB','T2N1M0','G2'),('TCGA-HV-AA8X-01A','PDAC','76','NA','no','Female','asian','IIB','T2N1MX','G2'),('TCGA-XN-A8T3-01A','PDAC','68','NA','no','Male','white','IB','T2N0M0','G2'),('TCGA-US-A77G-01A','PDAC','64','NA','yes','Male','white','IIB','T3N1MX','G2'),('TCGA-XD-AAUH-01A','PDAC','57','NA','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-2J-AABI-01A','PDAC','56','31','yes','Female','white','IIA','T3N0M0','G3'),('TCGA-3A-A9IR-01A','PDAC','64','15','yes','Female','white','IB','T2N0MX','G1'),('TCGA-IB-AAUS-01A','PDAC','84','65','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-LB-A9Q5-01A','PDAC','64','2','yes','Female','white','IIB','T3N1MX','G3'),('TCGA-FB-AAPS-01A','PDAC','63','10','no','Female','white','IIB','T2N1MX','G2'),('TCGA-HZ-7289-01A','PDAC','77','NA','no','Male','white','IIB','T3N1MX','G1'),('TCGA-FB-A4P6-01A','PDAC','55','25','yes','Male','white','IIB','T3N1MX','G1'),('TCGA-Q3-A5QY-01A','PDAC','58','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-3A-A9IC-01A','PDAC','61','NA','yes','Female','white','IIB','T3N1MX','G2'),('TCGA-IB-7646-01A','PDAC','61','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-HV-AA8V-01A','PDAC','50','NA','yes','Male','asian','IIB','T3N1MX','G3'),('TCGA-HZ-7922-01A','PDAC','61','NA','no','Female','white','IIB','T3N1MX','G1'),('TCGA-IB-AAUN-01A','PDAC','74','NA','yes','Female','white','IB','T2N0M0','G2'),('TCGA-YB-A89D-01A','PDAC','60','NA','no','Male','white','IIB','T3N1MX','G2'),('TCGA-HZ-7926-01A','PDAC','58','NA','no','Male','black or african american','IIB','T3N1MX','G1'),('TCGA-F2-7276-01A','PDAC','65','NA','no','Male','white','IIB','T3N1M0','G1'),('TCGA-FB-A4P5-01A','PDAC','70','40','no','Female','white','IIB','T3N1MX','G2'),('TCGA-IB-7651-01A','PDAC','65','NA','yes','Female','white','IIB','T3N1M0','G2'),('TCGA-HZ-8002-01A','PDAC','61','NA','NA','Male','white','IIB','T3N1MX','G2'),('TCGA-2L-AAQE-01A','PDAC','56','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-HZ-7920-01A','PDAC','72','NA','no','Male','white','IB','T2N0MX','G2'),('TCGA-IB-A5SO-01A','PDAC','72','NA','no','Male','white','IIB','T3N1M0','G2'),('TCGA-2J-AAB6-01A','PDAC','76','NA','no','Male','white','IIA','T3N0M0','G2'),('TCGA-HV-A5A4-01A','PDAC','72','NA','no','Female','asian','IIA','T3N0MX','G2'),('TCGA-HZ-7919-01A','PDAC','52','NA','no','Female','white','IIB','T3N1M0','G2'),('TCGA-FB-A5VM-01A','PDAC','75','47','yes','Male','white','IB','T2N0M0','G3'),('TCGA-2J-AAB1-01A','PDAC','66','26','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-RB-AA9M-01A','PDAC','43','NA','yes','Male','white','IIB','T1N1MX','G3'),('TCGA-IB-A6UF-01A','PDAC','63','NA','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-IB-A5SQ-01A','PDAC','57','NA','yes','Female','white','IB','T2N0M0','G2'),('TCGA-HV-A7OP-01A','PDAC','72','NA','yes','Male','asian','IIB','T3N1M0','GX'),('TCGA-3A-A9IN-01A','PDAC','62','NA','yes','Female','white','IB','T2NXMX','G2'),('TCGA-FB-AAPU-01A','PDAC','41','NA','yes','Female','asian','IIA','T3N0MX','G2'),('TCGA-IB-7891-01A','PDAC','50','29','yes','Female','white','IIB','T3N1M0','G1'),('TCGA-2J-AAB9-01A','PDAC','71','NA','yes','Female','white','IIB','T3N1M0','G1'),('TCGA-IB-7890-01A','PDAC','73','NA','yes','Male','not reported','IB','T2N0M0','G3'),('TCGA-IB-A5SS-01A','PDAC','64','NA','no','Female','white','IIB','T3N1M0','G3'),('TCGA-3A-A9IX-01A','PDAC','40','22','yes','Male','white','IA','T1N0MX','G2'),('TCGA-3A-A9IV-01A','PDAC','60','NA','yes','Female','white','IB','T2N0MX','G1'),('TCGA-3A-A9IU-01A','PDAC','66','NA','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-HZ-8005-01A','PDAC','81','NA','NA','Male','white','IIB','T3N1MX','G3'),('TCGA-2L-AAQA-01A','PDAC','77','NA','yes','Male','white','IIB','T3N1MX','G2'),('TCGA-IB-7886-01A','PDAC','80','NA','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-IB-AAUQ-01A','PDAC','50','23','yes','Male','white','IIB','T3N1M0','G2'),('TCGA-RL-AAAS-01A','PDAC','61','NA','yes','Female','white','IB','T2N0M0','G2'),('TCGA-3A-A9IS-01A','PDAC','67','NA','no','Male','white','IB','T2N0MX','G1'),('TCGA-IB-7645-01A','PDAC','44','NA','NA','Female','not reported','IIB','T3N1M0','G1'),('TCGA-3A-A9IH-01A','PDAC','66','NA','yes','Female','white','IA','T1N0MX','G2'),('TCGA-3A-A9IB-01A','PDAC','69','20','no','Female','white','IIB','T3N1MX','G3'),('TCGA-2J-AABU-01A','PDAC','57','NA','yes','Male','white','IIB','T3N1M0','G3'),('TCGA-3E-AAAY-01A','PDAC','68','NA','yes','Male','white','IIB','T3N1MX','G3'),('TCGA-IB-7654-01A','PDAC','80','NA','no','Male','white','IIB','T2N1M0','G2'),('TCGA-HZ-8637-01A','PDAC','77','NA','no','Female','white','IIB','T3N1MX','G3'),('TCGA-S4-A8RM-01A','PDAC','67','25','yes','Male','white','IIB','T3N1MX','G3');
/*!40000 ALTER TABLE `tcga` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2017-06-29 11:48:06
